2021-03-05 16:51:29,479 main.py:27  INFO Creating a new UserCollection
2021-03-05 16:51:29,481 main.py:36  INFO Createing a new UserStatusCollection
2021-03-05 16:51:29,483 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:51:29,483 main.py:86  INFO Saving collection in database
2021-03-05 16:51:29,484 main.py:59  ERROR User files must be in csv format
2021-03-05 16:51:29,486 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:51:29,486 main.py:72  ERROR Users need four parameters
2021-03-05 16:51:29,486 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:51:29,487 main.py:72  ERROR Users need four parameters
2021-03-05 16:51:29,487 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:51:29,487 main.py:72  ERROR Users need four parameters
2021-03-05 16:51:29,488 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:51:29,488 main.py:72  ERROR Users need four parameters
2021-03-05 16:51:29,488 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:51:29,488 main.py:72  ERROR Users need four parameters
2021-03-05 16:51:29,489 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:51:29,489 main.py:77  ERROR Users can not have empty fields
2021-03-05 16:51:29,490 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:51:29,490 main.py:77  ERROR Users can not have empty fields
2021-03-05 16:51:29,490 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:51:29,490 main.py:77  ERROR Users can not have empty fields
2021-03-05 16:51:29,490 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:51:29,490 main.py:77  ERROR Users can not have empty fields
2021-03-05 16:51:29,492 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:51:29,492 main.py:67  ERROR accounts.csv does not contain data
2021-03-05 16:51:29,493 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:51:29,493 main.py:140 INFO Saving collection in database
2021-03-05 16:51:29,495 main.py:113 ERROR UserStatus files must be in csv format
2021-03-05 16:51:29,497 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:51:29,497 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 16:51:29,497 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:51:29,497 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 16:51:29,497 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:51:29,497 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 16:51:29,498 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:51:29,498 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 16:51:29,498 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:51:29,498 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 16:51:29,499 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:51:29,499 main.py:131 ERROR UserStatus can not have empty fields
2021-03-05 16:51:29,500 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:51:29,500 main.py:131 ERROR UserStatus can not have empty fields
2021-03-05 16:51:29,500 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:51:29,500 main.py:131 ERROR UserStatus can not have empty fields
2021-03-05 16:51:29,501 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:51:29,501 main.py:121 ERROR status_updates.csv does not contain data
2021-03-05 16:51:29,503 main.py:158 INFO Adding a new User
2021-03-05 16:51:29,504 main.py:170 INFO Updating an User
2021-03-05 16:51:29,505 main.py:183 INFO Deleting an User
2021-03-05 16:51:29,507 main.py:197 INFO Searching for an User
2021-03-05 16:51:29,508 main.py:197 INFO Searching for an User
2021-03-05 16:51:29,515 main.py:215 INFO Adding an UserStatus
2021-03-05 16:51:29,516 main.py:227 INFO Updating an UserStatus
2021-03-05 16:51:29,518 main.py:239 INFO Deleting an UserStatus
2021-03-05 16:51:29,519 main.py:252 INFO Searching for an UserStatus
2021-03-05 16:51:29,520 main.py:252 INFO Searching for an UserStatus
2021-03-05 16:51:29,522 main.py:263 INFO Searching for matching UserStatuses
2021-03-05 16:51:29,523 main.py:272 INFO Searching for UserStatuses matching string
2021-03-05 16:51:30,026 main.py:27  INFO Creating a new UserCollection
2021-03-05 16:51:30,109 users.py:27  INFO New UserCollection instance created
2021-03-05 16:51:30,109 main.py:36  INFO Createing a new UserStatusCollection
2021-03-05 16:51:30,210 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:51:30,211 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:51:30,211 main.py:67  ERROR accounts.csv does not contain data
2021-03-05 16:51:30,211 main.py:158 INFO Adding a new User
2021-03-05 16:51:30,532 users.py:45  INFO User dave03 sucessfully added
2021-03-05 16:51:30,533 main.py:158 INFO Adding a new User
2021-03-05 16:51:30,535 users.py:43  ERROR User dave03 breaks database rule: E11000 duplicate key error collection: social.UserTable index: user_id_1 dup key: { user_id: "dave03" }, full error: {'index': 0, 'code': 11000, 'keyPattern': {'user_id': 1}, 'keyValue': {'user_id': 'dave03'}, 'errmsg': 'E11000 duplicate key error collection: social.UserTable index: user_id_1 dup key: { user_id: "dave03" }'}
2021-03-05 16:51:30,535 main.py:170 INFO Updating an User
2021-03-05 16:51:30,538 users.py:65  INFO User dave03 sucessfully modified
2021-03-05 16:51:30,538 main.py:170 INFO Updating an User
2021-03-05 16:51:30,540 users.py:61  ERROR User dave02 not found in database
2021-03-05 16:51:30,540 main.py:197 INFO Searching for an User
2021-03-05 16:51:30,542 users.py:96  INFO User dave03 sucessfully found
2021-03-05 16:51:30,542 main.py:197 INFO Searching for an User
2021-03-05 16:51:30,544 users.py:94  ERROR User dave02 not found in database
2021-03-05 16:51:30,544 main.py:183 INFO Deleting an User
2021-03-05 16:51:30,548 users.py:83  INFO User dave03 sucessfully deleted
2021-03-05 16:51:30,548 main.py:183 INFO Deleting an User
2021-03-05 16:51:30,550 users.py:78  ERROR User dave03 not found in database
2021-03-05 16:51:30,550 main.py:158 INFO Adding a new User
2021-03-05 16:51:30,552 users.py:45  INFO User dave03 sucessfully added
2021-03-05 16:51:30,552 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:51:30,552 main.py:121 ERROR status_updates.csv does not contain data
2021-03-05 16:51:30,552 main.py:215 INFO Adding an UserStatus
2021-03-05 16:51:30,626 user_status.py:45  INFO Status dave03_01 added to database
2021-03-05 16:51:30,627 main.py:215 INFO Adding an UserStatus
2021-03-05 16:51:30,630 user_status.py:43  ERROR Status dave03_01 already exists in database
2021-03-05 16:51:30,631 main.py:227 INFO Updating an UserStatus
2021-03-05 16:51:30,633 user_status.py:62  INFO Status dave03_01 sucessfully modified
2021-03-05 16:51:30,634 main.py:227 INFO Updating an UserStatus
2021-03-05 16:51:30,636 user_status.py:57  ERROR Status dave03_02 not found database
2021-03-05 16:51:30,636 main.py:252 INFO Searching for an UserStatus
2021-03-05 16:51:30,638 user_status.py:90  INFO Status dave03_01 sucessfully found
2021-03-05 16:51:30,638 main.py:252 INFO Searching for an UserStatus
2021-03-05 16:51:30,640 user_status.py:88  ERROR Status dave03 not found database
2021-03-05 16:51:30,640 main.py:239 INFO Deleting an UserStatus
2021-03-05 16:51:30,642 user_status.py:76  INFO Status dave03_01 sucessfully deleted
2021-03-05 16:51:30,643 main.py:239 INFO Deleting an UserStatus
2021-03-05 16:51:30,644 user_status.py:74  ERROR Status dave03_01 not found database
2021-03-05 16:51:30,644 main.py:215 INFO Adding an UserStatus
2021-03-05 16:51:30,648 user_status.py:45  INFO Status dave03_01 added to database
2021-03-05 16:51:30,648 main.py:183 INFO Deleting an User
2021-03-05 16:51:30,651 users.py:83  INFO User dave03 sucessfully deleted
2021-03-05 16:51:30,652 main.py:252 INFO Searching for an UserStatus
2021-03-05 16:51:30,653 user_status.py:88  ERROR Status dave03_01 not found database
2021-03-05 16:51:30,653 main.py:158 INFO Adding a new User
2021-03-05 16:51:30,654 users.py:104 ERROR User id ############################## is longer than 30
2021-03-05 16:51:30,654 main.py:158 INFO Adding a new User
2021-03-05 16:51:30,654 users.py:107 ERROR User name ############################## is longer than 30
2021-03-05 16:51:30,654 main.py:158 INFO Adding a new User
2021-03-05 16:51:30,654 users.py:112 ERROR User last name #################################################################################################### is longer than 100
2021-03-05 16:51:30,656 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:51:30,659 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:51:30,660 user_status.py:45  INFO Status rando_001 added to database
2021-03-05 16:51:30,663 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:51:30,664 user_status.py:43  ERROR Status rando_001 already exists in database
2021-03-05 16:51:30,665 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:51:30,666 user_status.py:39  ERROR User rando does not exist
2021-03-05 16:51:30,668 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:51:30,668 user_status.py:62  INFO Status rando_001 sucessfully modified
2021-03-05 16:51:30,670 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:51:30,671 user_status.py:57  ERROR Status rando_001 not found database
2021-03-05 16:51:30,673 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:51:30,674 user_status.py:76  INFO Status rando_001 sucessfully deleted
2021-03-05 16:51:30,676 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:51:30,676 user_status.py:74  ERROR Status rando_001 not found database
2021-03-05 16:51:30,678 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:51:30,679 user_status.py:90  INFO Status rando_001 sucessfully found
2021-03-05 16:51:30,680 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:51:30,681 user_status.py:88  ERROR Status rando_001 not found database
2021-03-05 16:51:30,682 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:51:30,683 user_status.py:99  INFO Found statuses posted by rando
2021-03-05 16:51:30,706 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:51:30,706 user_status.py:107 INFO Found statuses matching best
2021-03-05 16:51:30,732 users.py:27  INFO New UserCollection instance created
2021-03-05 16:51:30,734 users.py:27  INFO New UserCollection instance created
2021-03-05 16:51:30,735 users.py:45  INFO User rando sucessfully added
2021-03-05 16:51:30,737 users.py:27  INFO New UserCollection instance created
2021-03-05 16:51:30,737 users.py:43  ERROR User rando breaks database rule: ERR_MSG
2021-03-05 16:51:30,739 users.py:27  INFO New UserCollection instance created
2021-03-05 16:51:30,739 users.py:65  INFO User rando sucessfully modified
2021-03-05 16:51:30,741 users.py:27  INFO New UserCollection instance created
2021-03-05 16:51:30,742 users.py:61  ERROR User rando not found in database
2021-03-05 16:51:30,743 users.py:27  INFO New UserCollection instance created
2021-03-05 16:51:30,744 users.py:83  INFO User rando sucessfully deleted
2021-03-05 16:51:30,746 users.py:27  INFO New UserCollection instance created
2021-03-05 16:51:30,747 users.py:78  ERROR User rando not found in database
2021-03-05 16:51:30,749 users.py:27  INFO New UserCollection instance created
2021-03-05 16:51:30,749 users.py:96  INFO User rando sucessfully found
2021-03-05 16:51:30,751 users.py:27  INFO New UserCollection instance created
2021-03-05 16:51:30,751 users.py:94  ERROR User rando not found in database
2021-03-05 16:51:30,753 users.py:27  INFO New UserCollection instance created
2021-03-05 16:51:30,753 users.py:104 ERROR User id ############################## is longer than 30
2021-03-05 16:51:30,753 users.py:107 ERROR User name ############################## is longer than 30
2021-03-05 16:51:30,753 users.py:112 ERROR User last name #################################################################################################### is longer than 100
2021-03-05 16:55:04,417 main.py:27  INFO Creating a new UserCollection
2021-03-05 16:55:04,419 main.py:36  INFO Createing a new UserStatusCollection
2021-03-05 16:55:04,420 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:55:04,420 main.py:86  INFO Saving collection in database
2021-03-05 16:55:04,422 main.py:59  ERROR User files must be in csv format
2021-03-05 16:55:04,423 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:55:04,423 main.py:72  ERROR Users need four parameters
2021-03-05 16:55:04,423 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:55:04,424 main.py:72  ERROR Users need four parameters
2021-03-05 16:55:04,424 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:55:04,424 main.py:72  ERROR Users need four parameters
2021-03-05 16:55:04,424 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:55:04,424 main.py:72  ERROR Users need four parameters
2021-03-05 16:55:04,424 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:55:04,424 main.py:72  ERROR Users need four parameters
2021-03-05 16:55:04,426 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:55:04,426 main.py:77  ERROR Users can not have empty fields
2021-03-05 16:55:04,426 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:55:04,426 main.py:77  ERROR Users can not have empty fields
2021-03-05 16:55:04,426 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:55:04,426 main.py:77  ERROR Users can not have empty fields
2021-03-05 16:55:04,427 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:55:04,427 main.py:77  ERROR Users can not have empty fields
2021-03-05 16:55:04,428 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:55:04,428 main.py:67  ERROR accounts.csv does not contain data
2021-03-05 16:55:04,429 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:55:04,430 main.py:140 INFO Saving collection in database
2021-03-05 16:55:04,431 main.py:113 ERROR UserStatus files must be in csv format
2021-03-05 16:55:04,433 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:55:04,433 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 16:55:04,433 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:55:04,433 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 16:55:04,433 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:55:04,433 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 16:55:04,433 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:55:04,433 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 16:55:04,433 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:55:04,434 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 16:55:04,435 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:55:04,435 main.py:131 ERROR UserStatus can not have empty fields
2021-03-05 16:55:04,435 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:55:04,435 main.py:131 ERROR UserStatus can not have empty fields
2021-03-05 16:55:04,435 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:55:04,435 main.py:131 ERROR UserStatus can not have empty fields
2021-03-05 16:55:04,437 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:55:04,437 main.py:121 ERROR status_updates.csv does not contain data
2021-03-05 16:55:04,438 main.py:158 INFO Adding a new User
2021-03-05 16:55:04,439 main.py:170 INFO Updating an User
2021-03-05 16:55:04,441 main.py:183 INFO Deleting an User
2021-03-05 16:55:04,442 main.py:197 INFO Searching for an User
2021-03-05 16:55:04,443 main.py:197 INFO Searching for an User
2021-03-05 16:55:04,445 main.py:215 INFO Adding an UserStatus
2021-03-05 16:55:04,450 main.py:227 INFO Updating an UserStatus
2021-03-05 16:55:04,452 main.py:239 INFO Deleting an UserStatus
2021-03-05 16:55:04,453 main.py:252 INFO Searching for an UserStatus
2021-03-05 16:55:04,455 main.py:252 INFO Searching for an UserStatus
2021-03-05 16:55:04,456 main.py:263 INFO Searching for matching UserStatuses
2021-03-05 16:55:04,457 main.py:272 INFO Searching for UserStatuses matching string
2021-03-05 16:55:04,543 main.py:27  INFO Creating a new UserCollection
2021-03-05 16:55:04,611 users.py:27  INFO New UserCollection instance created
2021-03-05 16:55:04,611 main.py:36  INFO Createing a new UserStatusCollection
2021-03-05 16:55:04,711 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:55:04,711 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:55:04,711 main.py:67  ERROR accounts.csv does not contain data
2021-03-05 16:55:04,712 main.py:158 INFO Adding a new User
2021-03-05 16:55:05,058 users.py:45  INFO User dave03 sucessfully added
2021-03-05 16:55:05,058 main.py:158 INFO Adding a new User
2021-03-05 16:55:05,060 users.py:43  ERROR User dave03 breaks database rule: E11000 duplicate key error collection: social.UserTable index: user_id_1 dup key: { user_id: "dave03" }, full error: {'index': 0, 'code': 11000, 'keyPattern': {'user_id': 1}, 'keyValue': {'user_id': 'dave03'}, 'errmsg': 'E11000 duplicate key error collection: social.UserTable index: user_id_1 dup key: { user_id: "dave03" }'}
2021-03-05 16:55:05,061 main.py:170 INFO Updating an User
2021-03-05 16:55:05,064 users.py:65  INFO User dave03 sucessfully modified
2021-03-05 16:55:05,064 main.py:170 INFO Updating an User
2021-03-05 16:55:05,066 users.py:61  ERROR User dave02 not found in database
2021-03-05 16:55:05,066 main.py:197 INFO Searching for an User
2021-03-05 16:55:05,068 users.py:96  INFO User dave03 sucessfully found
2021-03-05 16:55:05,069 main.py:197 INFO Searching for an User
2021-03-05 16:55:05,071 users.py:94  ERROR User dave02 not found in database
2021-03-05 16:55:05,071 main.py:183 INFO Deleting an User
2021-03-05 16:55:05,074 users.py:83  INFO User dave03 sucessfully deleted
2021-03-05 16:55:05,075 main.py:183 INFO Deleting an User
2021-03-05 16:55:05,076 users.py:78  ERROR User dave03 not found in database
2021-03-05 16:55:05,077 main.py:158 INFO Adding a new User
2021-03-05 16:55:05,078 users.py:45  INFO User dave03 sucessfully added
2021-03-05 16:55:05,079 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:55:05,079 main.py:121 ERROR status_updates.csv does not contain data
2021-03-05 16:55:05,079 main.py:215 INFO Adding an UserStatus
2021-03-05 16:55:05,118 user_status.py:45  INFO Status dave03_01 added to database
2021-03-05 16:55:05,118 main.py:215 INFO Adding an UserStatus
2021-03-05 16:55:05,122 user_status.py:43  ERROR Status dave03_01 already exists in database
2021-03-05 16:55:05,122 main.py:227 INFO Updating an UserStatus
2021-03-05 16:55:05,125 user_status.py:62  INFO Status dave03_01 sucessfully modified
2021-03-05 16:55:05,125 main.py:227 INFO Updating an UserStatus
2021-03-05 16:55:05,127 user_status.py:57  ERROR Status dave03_02 not found database
2021-03-05 16:55:05,127 main.py:252 INFO Searching for an UserStatus
2021-03-05 16:55:05,130 user_status.py:90  INFO Status dave03_01 sucessfully found
2021-03-05 16:55:05,130 main.py:252 INFO Searching for an UserStatus
2021-03-05 16:55:05,132 user_status.py:88  ERROR Status dave03 not found database
2021-03-05 16:55:05,133 main.py:239 INFO Deleting an UserStatus
2021-03-05 16:55:05,135 user_status.py:76  INFO Status dave03_01 sucessfully deleted
2021-03-05 16:55:05,135 main.py:239 INFO Deleting an UserStatus
2021-03-05 16:55:05,137 user_status.py:74  ERROR Status dave03_01 not found database
2021-03-05 16:55:05,137 main.py:215 INFO Adding an UserStatus
2021-03-05 16:55:05,141 user_status.py:45  INFO Status dave03_01 added to database
2021-03-05 16:55:05,141 main.py:183 INFO Deleting an User
2021-03-05 16:55:05,145 users.py:83  INFO User dave03 sucessfully deleted
2021-03-05 16:55:05,145 main.py:252 INFO Searching for an UserStatus
2021-03-05 16:55:05,147 user_status.py:88  ERROR Status dave03_01 not found database
2021-03-05 16:55:05,147 main.py:158 INFO Adding a new User
2021-03-05 16:55:05,147 users.py:104 ERROR User id ############################## is longer than 30
2021-03-05 16:55:05,147 main.py:158 INFO Adding a new User
2021-03-05 16:55:05,148 users.py:107 ERROR User name ############################## is longer than 30
2021-03-05 16:55:05,148 main.py:158 INFO Adding a new User
2021-03-05 16:55:05,148 users.py:112 ERROR User last name #################################################################################################### is longer than 100
2021-03-05 16:55:05,150 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:55:05,152 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:55:05,154 user_status.py:45  INFO Status rando_001 added to database
2021-03-05 16:55:05,157 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:55:05,158 user_status.py:43  ERROR Status rando_001 already exists in database
2021-03-05 16:55:05,159 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:55:05,160 user_status.py:39  ERROR User rando does not exist
2021-03-05 16:55:05,162 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:55:05,162 user_status.py:62  INFO Status rando_001 sucessfully modified
2021-03-05 16:55:05,165 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:55:05,165 user_status.py:57  ERROR Status rando_001 not found database
2021-03-05 16:55:05,167 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:55:05,167 user_status.py:76  INFO Status rando_001 sucessfully deleted
2021-03-05 16:55:05,169 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:55:05,170 user_status.py:74  ERROR Status rando_001 not found database
2021-03-05 16:55:05,172 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:55:05,172 user_status.py:90  INFO Status rando_001 sucessfully found
2021-03-05 16:55:05,174 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:55:05,174 user_status.py:88  ERROR Status rando_001 not found database
2021-03-05 16:55:05,176 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:55:05,176 user_status.py:99  INFO Found statuses posted by rando
2021-03-05 16:55:05,199 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:55:05,200 user_status.py:107 INFO Found statuses matching best
2021-03-05 16:55:05,202 users.py:27  INFO New UserCollection instance created
2021-03-05 16:55:05,204 users.py:27  INFO New UserCollection instance created
2021-03-05 16:55:05,205 users.py:45  INFO User rando sucessfully added
2021-03-05 16:55:05,207 users.py:27  INFO New UserCollection instance created
2021-03-05 16:55:05,207 users.py:43  ERROR User rando breaks database rule: ERR_MSG
2021-03-05 16:55:05,209 users.py:27  INFO New UserCollection instance created
2021-03-05 16:55:05,209 users.py:65  INFO User rando sucessfully modified
2021-03-05 16:55:05,211 users.py:27  INFO New UserCollection instance created
2021-03-05 16:55:05,212 users.py:61  ERROR User rando not found in database
2021-03-05 16:55:05,213 users.py:27  INFO New UserCollection instance created
2021-03-05 16:55:05,215 users.py:83  INFO User rando sucessfully deleted
2021-03-05 16:55:05,216 users.py:27  INFO New UserCollection instance created
2021-03-05 16:55:05,217 users.py:78  ERROR User rando not found in database
2021-03-05 16:55:05,218 users.py:27  INFO New UserCollection instance created
2021-03-05 16:55:05,219 users.py:96  INFO User rando sucessfully found
2021-03-05 16:55:05,221 users.py:27  INFO New UserCollection instance created
2021-03-05 16:55:05,221 users.py:94  ERROR User rando not found in database
2021-03-05 16:55:05,223 users.py:27  INFO New UserCollection instance created
2021-03-05 16:55:05,223 users.py:104 ERROR User id ############################## is longer than 30
2021-03-05 16:55:05,223 users.py:107 ERROR User name ############################## is longer than 30
2021-03-05 16:55:05,223 users.py:112 ERROR User last name #################################################################################################### is longer than 100
2021-03-05 16:57:09,126 main.py:27  INFO Creating a new UserCollection
2021-03-05 16:57:09,127 main.py:36  INFO Createing a new UserStatusCollection
2021-03-05 16:57:09,129 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:57:09,129 main.py:86  INFO Saving collection in database
2021-03-05 16:57:09,131 main.py:59  ERROR User files must be in csv format
2021-03-05 16:57:09,132 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:57:09,132 main.py:72  ERROR Users need four parameters
2021-03-05 16:57:09,132 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:57:09,132 main.py:72  ERROR Users need four parameters
2021-03-05 16:57:09,132 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:57:09,133 main.py:72  ERROR Users need four parameters
2021-03-05 16:57:09,133 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:57:09,133 main.py:72  ERROR Users need four parameters
2021-03-05 16:57:09,133 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:57:09,133 main.py:72  ERROR Users need four parameters
2021-03-05 16:57:09,135 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:57:09,135 main.py:77  ERROR Users can not have empty fields
2021-03-05 16:57:09,135 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:57:09,135 main.py:77  ERROR Users can not have empty fields
2021-03-05 16:57:09,135 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:57:09,135 main.py:77  ERROR Users can not have empty fields
2021-03-05 16:57:09,135 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:57:09,135 main.py:77  ERROR Users can not have empty fields
2021-03-05 16:57:09,137 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:57:09,137 main.py:67  ERROR accounts.csv does not contain data
2021-03-05 16:57:09,138 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:57:09,138 main.py:140 INFO Saving collection in database
2021-03-05 16:57:09,140 main.py:113 ERROR UserStatus files must be in csv format
2021-03-05 16:57:09,141 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:57:09,141 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 16:57:09,141 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:57:09,142 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 16:57:09,142 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:57:09,142 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 16:57:09,142 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:57:09,142 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 16:57:09,142 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:57:09,142 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 16:57:09,144 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:57:09,144 main.py:131 ERROR UserStatus can not have empty fields
2021-03-05 16:57:09,144 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:57:09,144 main.py:131 ERROR UserStatus can not have empty fields
2021-03-05 16:57:09,144 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:57:09,144 main.py:131 ERROR UserStatus can not have empty fields
2021-03-05 16:57:09,145 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:57:09,146 main.py:121 ERROR status_updates.csv does not contain data
2021-03-05 16:57:09,147 main.py:158 INFO Adding a new User
2021-03-05 16:57:09,148 main.py:170 INFO Updating an User
2021-03-05 16:57:09,149 main.py:183 INFO Deleting an User
2021-03-05 16:57:09,151 main.py:197 INFO Searching for an User
2021-03-05 16:57:09,152 main.py:197 INFO Searching for an User
2021-03-05 16:57:09,153 main.py:215 INFO Adding an UserStatus
2021-03-05 16:57:09,159 main.py:227 INFO Updating an UserStatus
2021-03-05 16:57:09,160 main.py:239 INFO Deleting an UserStatus
2021-03-05 16:57:09,162 main.py:252 INFO Searching for an UserStatus
2021-03-05 16:57:09,163 main.py:252 INFO Searching for an UserStatus
2021-03-05 16:57:09,164 main.py:263 INFO Searching for matching UserStatuses
2021-03-05 16:57:09,165 main.py:272 INFO Searching for UserStatuses matching string
2021-03-05 16:57:09,252 main.py:27  INFO Creating a new UserCollection
2021-03-05 16:57:09,320 users.py:27  INFO New UserCollection instance created
2021-03-05 16:57:09,320 main.py:36  INFO Createing a new UserStatusCollection
2021-03-05 16:57:09,420 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:57:09,421 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:57:09,421 main.py:67  ERROR accounts.csv does not contain data
2021-03-05 16:57:09,421 main.py:158 INFO Adding a new User
2021-03-05 16:57:09,757 users.py:45  INFO User dave03 sucessfully added
2021-03-05 16:57:09,758 main.py:158 INFO Adding a new User
2021-03-05 16:57:09,760 users.py:43  ERROR User dave03 breaks database rule: E11000 duplicate key error collection: social.UserTable index: user_id_1 dup key: { user_id: "dave03" }, full error: {'index': 0, 'code': 11000, 'keyPattern': {'user_id': 1}, 'keyValue': {'user_id': 'dave03'}, 'errmsg': 'E11000 duplicate key error collection: social.UserTable index: user_id_1 dup key: { user_id: "dave03" }'}
2021-03-05 16:57:09,760 main.py:170 INFO Updating an User
2021-03-05 16:57:09,763 users.py:65  INFO User dave03 sucessfully modified
2021-03-05 16:57:09,763 main.py:170 INFO Updating an User
2021-03-05 16:57:09,765 users.py:61  ERROR User dave02 not found in database
2021-03-05 16:57:09,765 main.py:197 INFO Searching for an User
2021-03-05 16:57:09,767 users.py:96  INFO User dave03 sucessfully found
2021-03-05 16:57:09,767 main.py:197 INFO Searching for an User
2021-03-05 16:57:09,770 users.py:94  ERROR User dave02 not found in database
2021-03-05 16:57:09,770 main.py:183 INFO Deleting an User
2021-03-05 16:57:09,774 users.py:83  INFO User dave03 sucessfully deleted
2021-03-05 16:57:09,774 main.py:183 INFO Deleting an User
2021-03-05 16:57:09,776 users.py:78  ERROR User dave03 not found in database
2021-03-05 16:57:09,776 main.py:158 INFO Adding a new User
2021-03-05 16:57:09,778 users.py:45  INFO User dave03 sucessfully added
2021-03-05 16:57:09,778 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:57:09,779 main.py:121 ERROR status_updates.csv does not contain data
2021-03-05 16:57:09,779 main.py:215 INFO Adding an UserStatus
2021-03-05 16:57:09,836 user_status.py:45  INFO Status dave03_01 added to database
2021-03-05 16:57:09,836 main.py:215 INFO Adding an UserStatus
2021-03-05 16:57:09,840 user_status.py:43  ERROR Status dave03_01 already exists in database
2021-03-05 16:57:09,840 main.py:227 INFO Updating an UserStatus
2021-03-05 16:57:09,842 user_status.py:62  INFO Status dave03_01 sucessfully modified
2021-03-05 16:57:09,842 main.py:227 INFO Updating an UserStatus
2021-03-05 16:57:09,844 user_status.py:57  ERROR Status dave03_02 not found database
2021-03-05 16:57:09,845 main.py:252 INFO Searching for an UserStatus
2021-03-05 16:57:09,847 user_status.py:90  INFO Status dave03_01 sucessfully found
2021-03-05 16:57:09,848 main.py:252 INFO Searching for an UserStatus
2021-03-05 16:57:09,849 user_status.py:88  ERROR Status dave03 not found database
2021-03-05 16:57:09,850 main.py:239 INFO Deleting an UserStatus
2021-03-05 16:57:09,851 user_status.py:76  INFO Status dave03_01 sucessfully deleted
2021-03-05 16:57:09,851 main.py:239 INFO Deleting an UserStatus
2021-03-05 16:57:09,854 user_status.py:74  ERROR Status dave03_01 not found database
2021-03-05 16:57:09,854 main.py:215 INFO Adding an UserStatus
2021-03-05 16:57:09,858 user_status.py:45  INFO Status dave03_01 added to database
2021-03-05 16:57:09,858 main.py:183 INFO Deleting an User
2021-03-05 16:57:09,862 users.py:83  INFO User dave03 sucessfully deleted
2021-03-05 16:57:09,862 main.py:252 INFO Searching for an UserStatus
2021-03-05 16:57:09,864 user_status.py:88  ERROR Status dave03_01 not found database
2021-03-05 16:57:09,864 main.py:158 INFO Adding a new User
2021-03-05 16:57:09,864 users.py:104 ERROR User id ############################## is longer than 30
2021-03-05 16:57:09,864 main.py:158 INFO Adding a new User
2021-03-05 16:57:09,864 users.py:107 ERROR User name ############################## is longer than 30
2021-03-05 16:57:09,864 main.py:158 INFO Adding a new User
2021-03-05 16:57:09,864 users.py:112 ERROR User last name #################################################################################################### is longer than 100
2021-03-05 16:57:09,867 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:57:09,869 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:57:09,871 user_status.py:45  INFO Status rando_001 added to database
2021-03-05 16:57:09,874 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:57:09,874 user_status.py:43  ERROR Status rando_001 already exists in database
2021-03-05 16:57:09,876 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:57:09,877 user_status.py:39  ERROR User rando does not exist
2021-03-05 16:57:09,878 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:57:09,879 user_status.py:62  INFO Status rando_001 sucessfully modified
2021-03-05 16:57:09,881 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:57:09,882 user_status.py:57  ERROR Status rando_001 not found database
2021-03-05 16:57:09,883 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:57:09,884 user_status.py:76  INFO Status rando_001 sucessfully deleted
2021-03-05 16:57:09,886 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:57:09,886 user_status.py:74  ERROR Status rando_001 not found database
2021-03-05 16:57:09,888 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:57:09,889 user_status.py:90  INFO Status rando_001 sucessfully found
2021-03-05 16:57:09,890 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:57:09,891 user_status.py:88  ERROR Status rando_001 not found database
2021-03-05 16:57:09,893 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:57:09,893 user_status.py:99  INFO Found statuses posted by rando
2021-03-05 16:57:09,895 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:57:09,896 user_status.py:107 INFO Found statuses matching best
2021-03-05 16:57:09,898 users.py:27  INFO New UserCollection instance created
2021-03-05 16:57:09,900 users.py:27  INFO New UserCollection instance created
2021-03-05 16:57:09,900 users.py:45  INFO User rando sucessfully added
2021-03-05 16:57:09,902 users.py:27  INFO New UserCollection instance created
2021-03-05 16:57:09,903 users.py:43  ERROR User rando breaks database rule: ERR_MSG
2021-03-05 16:57:09,905 users.py:27  INFO New UserCollection instance created
2021-03-05 16:57:09,905 users.py:65  INFO User rando sucessfully modified
2021-03-05 16:57:09,907 users.py:27  INFO New UserCollection instance created
2021-03-05 16:57:09,907 users.py:61  ERROR User rando not found in database
2021-03-05 16:57:09,909 users.py:27  INFO New UserCollection instance created
2021-03-05 16:57:09,910 users.py:83  INFO User rando sucessfully deleted
2021-03-05 16:57:09,912 users.py:27  INFO New UserCollection instance created
2021-03-05 16:57:09,912 users.py:78  ERROR User rando not found in database
2021-03-05 16:57:09,914 users.py:27  INFO New UserCollection instance created
2021-03-05 16:57:09,914 users.py:96  INFO User rando sucessfully found
2021-03-05 16:57:09,916 users.py:27  INFO New UserCollection instance created
2021-03-05 16:57:09,917 users.py:94  ERROR User rando not found in database
2021-03-05 16:57:09,918 users.py:27  INFO New UserCollection instance created
2021-03-05 16:57:09,919 users.py:104 ERROR User id ############################## is longer than 30
2021-03-05 16:57:09,919 users.py:107 ERROR User name ############################## is longer than 30
2021-03-05 16:57:09,919 users.py:112 ERROR User last name #################################################################################################### is longer than 100
2021-03-05 16:58:17,633 main.py:27  INFO Creating a new UserCollection
2021-03-05 16:58:17,634 main.py:36  INFO Createing a new UserStatusCollection
2021-03-05 16:58:17,636 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:58:17,636 main.py:86  INFO Saving collection in database
2021-03-05 16:58:17,637 main.py:59  ERROR User files must be in csv format
2021-03-05 16:58:17,639 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:58:17,639 main.py:72  ERROR Users need four parameters
2021-03-05 16:58:17,639 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:58:17,639 main.py:72  ERROR Users need four parameters
2021-03-05 16:58:17,639 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:58:17,639 main.py:72  ERROR Users need four parameters
2021-03-05 16:58:17,639 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:58:17,640 main.py:72  ERROR Users need four parameters
2021-03-05 16:58:17,640 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:58:17,640 main.py:72  ERROR Users need four parameters
2021-03-05 16:58:17,641 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:58:17,641 main.py:77  ERROR Users can not have empty fields
2021-03-05 16:58:17,641 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:58:17,642 main.py:77  ERROR Users can not have empty fields
2021-03-05 16:58:17,642 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:58:17,642 main.py:77  ERROR Users can not have empty fields
2021-03-05 16:58:17,642 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:58:17,642 main.py:77  ERROR Users can not have empty fields
2021-03-05 16:58:17,644 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:58:17,644 main.py:67  ERROR accounts.csv does not contain data
2021-03-05 16:58:17,645 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:58:17,645 main.py:140 INFO Saving collection in database
2021-03-05 16:58:17,647 main.py:113 ERROR UserStatus files must be in csv format
2021-03-05 16:58:17,648 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:58:17,648 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 16:58:17,648 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:58:17,648 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 16:58:17,648 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:58:17,648 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 16:58:17,649 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:58:17,649 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 16:58:17,649 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:58:17,649 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 16:58:17,650 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:58:17,650 main.py:131 ERROR UserStatus can not have empty fields
2021-03-05 16:58:17,651 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:58:17,651 main.py:131 ERROR UserStatus can not have empty fields
2021-03-05 16:58:17,651 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:58:17,651 main.py:131 ERROR UserStatus can not have empty fields
2021-03-05 16:58:17,652 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:58:17,652 main.py:121 ERROR status_updates.csv does not contain data
2021-03-05 16:58:17,654 main.py:158 INFO Adding a new User
2021-03-05 16:58:17,655 main.py:170 INFO Updating an User
2021-03-05 16:58:17,656 main.py:183 INFO Deleting an User
2021-03-05 16:58:17,657 main.py:197 INFO Searching for an User
2021-03-05 16:58:17,658 main.py:197 INFO Searching for an User
2021-03-05 16:58:17,660 main.py:215 INFO Adding an UserStatus
2021-03-05 16:58:17,661 main.py:227 INFO Updating an UserStatus
2021-03-05 16:58:17,667 main.py:239 INFO Deleting an UserStatus
2021-03-05 16:58:17,669 main.py:252 INFO Searching for an UserStatus
2021-03-05 16:58:17,670 main.py:252 INFO Searching for an UserStatus
2021-03-05 16:58:17,671 main.py:263 INFO Searching for matching UserStatuses
2021-03-05 16:58:17,672 main.py:272 INFO Searching for UserStatuses matching string
2021-03-05 16:58:17,758 main.py:27  INFO Creating a new UserCollection
2021-03-05 16:58:17,837 users.py:27  INFO New UserCollection instance created
2021-03-05 16:58:17,837 main.py:36  INFO Createing a new UserStatusCollection
2021-03-05 16:58:17,926 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:58:17,926 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:58:17,926 main.py:67  ERROR accounts.csv does not contain data
2021-03-05 16:58:17,927 main.py:158 INFO Adding a new User
2021-03-05 16:58:18,272 users.py:45  INFO User dave03 sucessfully added
2021-03-05 16:58:18,272 main.py:158 INFO Adding a new User
2021-03-05 16:58:18,274 users.py:43  ERROR User dave03 breaks database rule: E11000 duplicate key error collection: social.UserTable index: user_id_1 dup key: { user_id: "dave03" }, full error: {'index': 0, 'code': 11000, 'keyPattern': {'user_id': 1}, 'keyValue': {'user_id': 'dave03'}, 'errmsg': 'E11000 duplicate key error collection: social.UserTable index: user_id_1 dup key: { user_id: "dave03" }'}
2021-03-05 16:58:18,274 main.py:170 INFO Updating an User
2021-03-05 16:58:18,277 users.py:65  INFO User dave03 sucessfully modified
2021-03-05 16:58:18,277 main.py:170 INFO Updating an User
2021-03-05 16:58:18,279 users.py:61  ERROR User dave02 not found in database
2021-03-05 16:58:18,280 main.py:197 INFO Searching for an User
2021-03-05 16:58:18,281 users.py:96  INFO User dave03 sucessfully found
2021-03-05 16:58:18,281 main.py:197 INFO Searching for an User
2021-03-05 16:58:18,283 users.py:94  ERROR User dave02 not found in database
2021-03-05 16:58:18,283 main.py:183 INFO Deleting an User
2021-03-05 16:58:18,287 users.py:83  INFO User dave03 sucessfully deleted
2021-03-05 16:58:18,287 main.py:183 INFO Deleting an User
2021-03-05 16:58:18,288 users.py:78  ERROR User dave03 not found in database
2021-03-05 16:58:18,288 main.py:158 INFO Adding a new User
2021-03-05 16:58:18,290 users.py:45  INFO User dave03 sucessfully added
2021-03-05 16:58:18,290 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:58:18,290 main.py:121 ERROR status_updates.csv does not contain data
2021-03-05 16:58:18,291 main.py:215 INFO Adding an UserStatus
2021-03-05 16:58:18,350 user_status.py:45  INFO Status dave03_01 added to database
2021-03-05 16:58:18,351 main.py:215 INFO Adding an UserStatus
2021-03-05 16:58:18,355 user_status.py:43  ERROR Status dave03_01 already exists in database
2021-03-05 16:58:18,355 main.py:227 INFO Updating an UserStatus
2021-03-05 16:58:18,357 user_status.py:62  INFO Status dave03_01 sucessfully modified
2021-03-05 16:58:18,357 main.py:227 INFO Updating an UserStatus
2021-03-05 16:58:18,359 user_status.py:57  ERROR Status dave03_02 not found database
2021-03-05 16:58:18,359 main.py:252 INFO Searching for an UserStatus
2021-03-05 16:58:18,361 user_status.py:90  INFO Status dave03_01 sucessfully found
2021-03-05 16:58:18,361 main.py:252 INFO Searching for an UserStatus
2021-03-05 16:58:18,363 user_status.py:88  ERROR Status dave03 not found database
2021-03-05 16:58:18,363 main.py:239 INFO Deleting an UserStatus
2021-03-05 16:58:18,365 user_status.py:76  INFO Status dave03_01 sucessfully deleted
2021-03-05 16:58:18,365 main.py:239 INFO Deleting an UserStatus
2021-03-05 16:58:18,367 user_status.py:74  ERROR Status dave03_01 not found database
2021-03-05 16:58:18,367 main.py:215 INFO Adding an UserStatus
2021-03-05 16:58:18,371 user_status.py:45  INFO Status dave03_01 added to database
2021-03-05 16:58:18,371 main.py:183 INFO Deleting an User
2021-03-05 16:58:18,374 users.py:83  INFO User dave03 sucessfully deleted
2021-03-05 16:58:18,374 main.py:252 INFO Searching for an UserStatus
2021-03-05 16:58:18,376 user_status.py:88  ERROR Status dave03_01 not found database
2021-03-05 16:58:18,376 main.py:158 INFO Adding a new User
2021-03-05 16:58:18,376 users.py:104 ERROR User id ############################## is longer than 30
2021-03-05 16:58:18,376 main.py:158 INFO Adding a new User
2021-03-05 16:58:18,377 users.py:107 ERROR User name ############################## is longer than 30
2021-03-05 16:58:18,377 main.py:158 INFO Adding a new User
2021-03-05 16:58:18,377 users.py:112 ERROR User last name #################################################################################################### is longer than 100
2021-03-05 16:58:18,379 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:58:18,381 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:58:18,383 user_status.py:45  INFO Status rando_001 added to database
2021-03-05 16:58:18,386 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:58:18,387 user_status.py:43  ERROR Status rando_001 already exists in database
2021-03-05 16:58:18,388 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:58:18,389 user_status.py:39  ERROR User rando does not exist
2021-03-05 16:58:18,391 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:58:18,391 user_status.py:62  INFO Status rando_001 sucessfully modified
2021-03-05 16:58:18,394 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:58:18,394 user_status.py:57  ERROR Status rando_001 not found database
2021-03-05 16:58:18,396 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:58:18,396 user_status.py:76  INFO Status rando_001 sucessfully deleted
2021-03-05 16:58:18,398 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:58:18,399 user_status.py:74  ERROR Status rando_001 not found database
2021-03-05 16:58:18,400 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:58:18,401 user_status.py:90  INFO Status rando_001 sucessfully found
2021-03-05 16:58:18,403 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:58:18,403 user_status.py:88  ERROR Status rando_001 not found database
2021-03-05 16:58:18,405 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:58:18,406 user_status.py:99  INFO Found statuses posted by rando
2021-03-05 16:58:18,430 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:58:18,431 user_status.py:107 INFO Found statuses matching best
2021-03-05 16:58:18,433 users.py:27  INFO New UserCollection instance created
2021-03-05 16:58:18,435 users.py:27  INFO New UserCollection instance created
2021-03-05 16:58:18,436 users.py:45  INFO User rando sucessfully added
2021-03-05 16:58:18,438 users.py:27  INFO New UserCollection instance created
2021-03-05 16:58:18,439 users.py:43  ERROR User rando breaks database rule: ERR_MSG
2021-03-05 16:58:18,441 users.py:27  INFO New UserCollection instance created
2021-03-05 16:58:18,441 users.py:65  INFO User rando sucessfully modified
2021-03-05 16:58:18,444 users.py:27  INFO New UserCollection instance created
2021-03-05 16:58:18,444 users.py:61  ERROR User rando not found in database
2021-03-05 16:58:18,447 users.py:27  INFO New UserCollection instance created
2021-03-05 16:58:18,448 users.py:83  INFO User rando sucessfully deleted
2021-03-05 16:58:18,449 users.py:27  INFO New UserCollection instance created
2021-03-05 16:58:18,450 users.py:78  ERROR User rando not found in database
2021-03-05 16:58:18,451 users.py:27  INFO New UserCollection instance created
2021-03-05 16:58:18,452 users.py:96  INFO User rando sucessfully found
2021-03-05 16:58:18,454 users.py:27  INFO New UserCollection instance created
2021-03-05 16:58:18,455 users.py:94  ERROR User rando not found in database
2021-03-05 16:58:18,457 users.py:27  INFO New UserCollection instance created
2021-03-05 16:58:18,457 users.py:104 ERROR User id ############################## is longer than 30
2021-03-05 16:58:18,457 users.py:107 ERROR User name ############################## is longer than 30
2021-03-05 16:58:18,457 users.py:112 ERROR User last name #################################################################################################### is longer than 100
2021-03-05 16:59:02,363 main.py:27  INFO Creating a new UserCollection
2021-03-05 16:59:02,365 main.py:36  INFO Createing a new UserStatusCollection
2021-03-05 16:59:02,366 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:59:02,366 main.py:86  INFO Saving collection in database
2021-03-05 16:59:02,368 main.py:59  ERROR User files must be in csv format
2021-03-05 16:59:02,369 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:59:02,369 main.py:72  ERROR Users need four parameters
2021-03-05 16:59:02,369 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:59:02,370 main.py:72  ERROR Users need four parameters
2021-03-05 16:59:02,370 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:59:02,370 main.py:72  ERROR Users need four parameters
2021-03-05 16:59:02,370 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:59:02,370 main.py:72  ERROR Users need four parameters
2021-03-05 16:59:02,370 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:59:02,370 main.py:72  ERROR Users need four parameters
2021-03-05 16:59:02,372 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:59:02,372 main.py:77  ERROR Users can not have empty fields
2021-03-05 16:59:02,372 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:59:02,372 main.py:77  ERROR Users can not have empty fields
2021-03-05 16:59:02,372 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:59:02,372 main.py:77  ERROR Users can not have empty fields
2021-03-05 16:59:02,373 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:59:02,373 main.py:77  ERROR Users can not have empty fields
2021-03-05 16:59:02,374 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:59:02,374 main.py:67  ERROR accounts.csv does not contain data
2021-03-05 16:59:02,375 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:59:02,375 main.py:140 INFO Saving collection in database
2021-03-05 16:59:02,377 main.py:113 ERROR UserStatus files must be in csv format
2021-03-05 16:59:02,378 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:59:02,378 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 16:59:02,379 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:59:02,379 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 16:59:02,379 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:59:02,379 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 16:59:02,379 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:59:02,379 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 16:59:02,379 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:59:02,379 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 16:59:02,381 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:59:02,381 main.py:131 ERROR UserStatus can not have empty fields
2021-03-05 16:59:02,381 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:59:02,381 main.py:131 ERROR UserStatus can not have empty fields
2021-03-05 16:59:02,381 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:59:02,381 main.py:131 ERROR UserStatus can not have empty fields
2021-03-05 16:59:02,383 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:59:02,383 main.py:121 ERROR status_updates.csv does not contain data
2021-03-05 16:59:02,384 main.py:158 INFO Adding a new User
2021-03-05 16:59:02,385 main.py:170 INFO Updating an User
2021-03-05 16:59:02,386 main.py:183 INFO Deleting an User
2021-03-05 16:59:02,388 main.py:197 INFO Searching for an User
2021-03-05 16:59:02,389 main.py:197 INFO Searching for an User
2021-03-05 16:59:02,390 main.py:215 INFO Adding an UserStatus
2021-03-05 16:59:02,397 main.py:227 INFO Updating an UserStatus
2021-03-05 16:59:02,398 main.py:239 INFO Deleting an UserStatus
2021-03-05 16:59:02,399 main.py:252 INFO Searching for an UserStatus
2021-03-05 16:59:02,401 main.py:252 INFO Searching for an UserStatus
2021-03-05 16:59:02,402 main.py:263 INFO Searching for matching UserStatuses
2021-03-05 16:59:02,403 main.py:272 INFO Searching for UserStatuses matching string
2021-03-05 16:59:02,487 main.py:27  INFO Creating a new UserCollection
2021-03-05 16:59:02,554 users.py:27  INFO New UserCollection instance created
2021-03-05 16:59:02,554 main.py:36  INFO Createing a new UserStatusCollection
2021-03-05 16:59:02,644 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:59:02,644 main.py:62  INFO Loading users from accounts.csv
2021-03-05 16:59:02,644 main.py:67  ERROR accounts.csv does not contain data
2021-03-05 16:59:02,645 main.py:158 INFO Adding a new User
2021-03-05 16:59:02,995 users.py:45  INFO User dave03 sucessfully added
2021-03-05 16:59:02,995 main.py:158 INFO Adding a new User
2021-03-05 16:59:02,997 users.py:43  ERROR User dave03 breaks database rule: E11000 duplicate key error collection: social.UserTable index: user_id_1 dup key: { user_id: "dave03" }, full error: {'index': 0, 'code': 11000, 'keyPattern': {'user_id': 1}, 'keyValue': {'user_id': 'dave03'}, 'errmsg': 'E11000 duplicate key error collection: social.UserTable index: user_id_1 dup key: { user_id: "dave03" }'}
2021-03-05 16:59:02,998 main.py:170 INFO Updating an User
2021-03-05 16:59:03,001 users.py:65  INFO User dave03 sucessfully modified
2021-03-05 16:59:03,001 main.py:170 INFO Updating an User
2021-03-05 16:59:03,003 users.py:61  ERROR User dave02 not found in database
2021-03-05 16:59:03,003 main.py:197 INFO Searching for an User
2021-03-05 16:59:03,006 users.py:96  INFO User dave03 sucessfully found
2021-03-05 16:59:03,006 main.py:197 INFO Searching for an User
2021-03-05 16:59:03,009 users.py:94  ERROR User dave02 not found in database
2021-03-05 16:59:03,009 main.py:183 INFO Deleting an User
2021-03-05 16:59:03,013 users.py:83  INFO User dave03 sucessfully deleted
2021-03-05 16:59:03,013 main.py:183 INFO Deleting an User
2021-03-05 16:59:03,015 users.py:78  ERROR User dave03 not found in database
2021-03-05 16:59:03,015 main.py:158 INFO Adding a new User
2021-03-05 16:59:03,017 users.py:45  INFO User dave03 sucessfully added
2021-03-05 16:59:03,018 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 16:59:03,018 main.py:121 ERROR status_updates.csv does not contain data
2021-03-05 16:59:03,018 main.py:215 INFO Adding an UserStatus
2021-03-05 16:59:03,071 user_status.py:45  INFO Status dave03_01 added to database
2021-03-05 16:59:03,072 main.py:215 INFO Adding an UserStatus
2021-03-05 16:59:03,075 user_status.py:43  ERROR Status dave03_01 already exists in database
2021-03-05 16:59:03,076 main.py:227 INFO Updating an UserStatus
2021-03-05 16:59:03,078 user_status.py:62  INFO Status dave03_01 sucessfully modified
2021-03-05 16:59:03,078 main.py:227 INFO Updating an UserStatus
2021-03-05 16:59:03,081 user_status.py:57  ERROR Status dave03_02 not found database
2021-03-05 16:59:03,081 main.py:252 INFO Searching for an UserStatus
2021-03-05 16:59:03,083 user_status.py:90  INFO Status dave03_01 sucessfully found
2021-03-05 16:59:03,084 main.py:252 INFO Searching for an UserStatus
2021-03-05 16:59:03,086 user_status.py:88  ERROR Status dave03 not found database
2021-03-05 16:59:03,086 main.py:239 INFO Deleting an UserStatus
2021-03-05 16:59:03,088 user_status.py:76  INFO Status dave03_01 sucessfully deleted
2021-03-05 16:59:03,088 main.py:239 INFO Deleting an UserStatus
2021-03-05 16:59:03,090 user_status.py:74  ERROR Status dave03_01 not found database
2021-03-05 16:59:03,091 main.py:215 INFO Adding an UserStatus
2021-03-05 16:59:03,095 user_status.py:45  INFO Status dave03_01 added to database
2021-03-05 16:59:03,095 main.py:183 INFO Deleting an User
2021-03-05 16:59:03,099 users.py:83  INFO User dave03 sucessfully deleted
2021-03-05 16:59:03,099 main.py:252 INFO Searching for an UserStatus
2021-03-05 16:59:03,101 user_status.py:88  ERROR Status dave03_01 not found database
2021-03-05 16:59:03,102 main.py:158 INFO Adding a new User
2021-03-05 16:59:03,102 users.py:104 ERROR User id ############################## is longer than 30
2021-03-05 16:59:03,102 main.py:158 INFO Adding a new User
2021-03-05 16:59:03,102 users.py:107 ERROR User name ############################## is longer than 30
2021-03-05 16:59:03,102 main.py:158 INFO Adding a new User
2021-03-05 16:59:03,102 users.py:112 ERROR User last name #################################################################################################### is longer than 100
2021-03-05 16:59:03,105 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:59:03,107 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:59:03,108 user_status.py:45  INFO Status rando_001 added to database
2021-03-05 16:59:03,111 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:59:03,112 user_status.py:43  ERROR Status rando_001 already exists in database
2021-03-05 16:59:03,114 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:59:03,115 user_status.py:39  ERROR User rando does not exist
2021-03-05 16:59:03,116 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:59:03,117 user_status.py:62  INFO Status rando_001 sucessfully modified
2021-03-05 16:59:03,119 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:59:03,120 user_status.py:57  ERROR Status rando_001 not found database
2021-03-05 16:59:03,121 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:59:03,122 user_status.py:76  INFO Status rando_001 sucessfully deleted
2021-03-05 16:59:03,124 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:59:03,124 user_status.py:74  ERROR Status rando_001 not found database
2021-03-05 16:59:03,126 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:59:03,127 user_status.py:90  INFO Status rando_001 sucessfully found
2021-03-05 16:59:03,128 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:59:03,129 user_status.py:88  ERROR Status rando_001 not found database
2021-03-05 16:59:03,130 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:59:03,131 user_status.py:99  INFO Found statuses posted by rando
2021-03-05 16:59:03,133 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 16:59:03,134 user_status.py:107 INFO Found statuses matching best
2021-03-05 16:59:03,135 users.py:27  INFO New UserCollection instance created
2021-03-05 16:59:03,138 users.py:27  INFO New UserCollection instance created
2021-03-05 16:59:03,138 users.py:45  INFO User rando sucessfully added
2021-03-05 16:59:03,140 users.py:27  INFO New UserCollection instance created
2021-03-05 16:59:03,140 users.py:43  ERROR User rando breaks database rule: ERR_MSG
2021-03-05 16:59:03,142 users.py:27  INFO New UserCollection instance created
2021-03-05 16:59:03,143 users.py:65  INFO User rando sucessfully modified
2021-03-05 16:59:03,145 users.py:27  INFO New UserCollection instance created
2021-03-05 16:59:03,145 users.py:61  ERROR User rando not found in database
2021-03-05 16:59:03,147 users.py:27  INFO New UserCollection instance created
2021-03-05 16:59:03,148 users.py:83  INFO User rando sucessfully deleted
2021-03-05 16:59:03,150 users.py:27  INFO New UserCollection instance created
2021-03-05 16:59:03,150 users.py:78  ERROR User rando not found in database
2021-03-05 16:59:03,152 users.py:27  INFO New UserCollection instance created
2021-03-05 16:59:03,153 users.py:96  INFO User rando sucessfully found
2021-03-05 16:59:03,154 users.py:27  INFO New UserCollection instance created
2021-03-05 16:59:03,155 users.py:94  ERROR User rando not found in database
2021-03-05 16:59:03,156 users.py:27  INFO New UserCollection instance created
2021-03-05 16:59:03,157 users.py:104 ERROR User id ############################## is longer than 30
2021-03-05 16:59:03,157 users.py:107 ERROR User name ############################## is longer than 30
2021-03-05 16:59:03,157 users.py:112 ERROR User last name #################################################################################################### is longer than 100
2021-03-05 17:01:38,933 main.py:27  INFO Creating a new UserCollection
2021-03-05 17:01:38,934 main.py:36  INFO Createing a new UserStatusCollection
2021-03-05 17:01:38,936 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:01:38,936 main.py:86  INFO Saving collection in database
2021-03-05 17:01:38,938 main.py:59  ERROR User files must be in csv format
2021-03-05 17:01:38,939 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:01:38,939 main.py:72  ERROR Users need four parameters
2021-03-05 17:01:38,940 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:01:38,940 main.py:72  ERROR Users need four parameters
2021-03-05 17:01:38,940 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:01:38,940 main.py:72  ERROR Users need four parameters
2021-03-05 17:01:38,940 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:01:38,940 main.py:72  ERROR Users need four parameters
2021-03-05 17:01:38,940 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:01:38,940 main.py:72  ERROR Users need four parameters
2021-03-05 17:01:38,942 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:01:38,942 main.py:77  ERROR Users can not have empty fields
2021-03-05 17:01:38,942 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:01:38,942 main.py:77  ERROR Users can not have empty fields
2021-03-05 17:01:38,942 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:01:38,942 main.py:77  ERROR Users can not have empty fields
2021-03-05 17:01:38,942 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:01:38,943 main.py:77  ERROR Users can not have empty fields
2021-03-05 17:01:38,944 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:01:38,944 main.py:67  ERROR accounts.csv does not contain data
2021-03-05 17:01:38,945 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:01:38,945 main.py:140 INFO Saving collection in database
2021-03-05 17:01:38,947 main.py:113 ERROR UserStatus files must be in csv format
2021-03-05 17:01:38,949 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:01:38,949 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 17:01:38,949 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:01:38,949 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 17:01:38,949 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:01:38,949 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 17:01:38,950 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:01:38,950 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 17:01:38,950 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:01:38,950 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 17:01:38,951 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:01:38,951 main.py:131 ERROR UserStatus can not have empty fields
2021-03-05 17:01:38,951 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:01:38,951 main.py:131 ERROR UserStatus can not have empty fields
2021-03-05 17:01:38,952 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:01:38,952 main.py:131 ERROR UserStatus can not have empty fields
2021-03-05 17:01:38,958 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:01:38,958 main.py:121 ERROR status_updates.csv does not contain data
2021-03-05 17:01:38,959 main.py:158 INFO Adding a new User
2021-03-05 17:01:38,960 main.py:170 INFO Updating an User
2021-03-05 17:01:38,962 main.py:183 INFO Deleting an User
2021-03-05 17:01:38,963 main.py:197 INFO Searching for an User
2021-03-05 17:01:38,964 main.py:197 INFO Searching for an User
2021-03-05 17:01:38,965 main.py:215 INFO Adding an UserStatus
2021-03-05 17:01:38,966 main.py:227 INFO Updating an UserStatus
2021-03-05 17:01:38,968 main.py:239 INFO Deleting an UserStatus
2021-03-05 17:01:38,969 main.py:252 INFO Searching for an UserStatus
2021-03-05 17:01:38,970 main.py:252 INFO Searching for an UserStatus
2021-03-05 17:01:38,971 main.py:263 INFO Searching for matching UserStatuses
2021-03-05 17:01:38,973 main.py:272 INFO Searching for UserStatuses matching string
2021-03-05 17:01:39,063 main.py:27  INFO Creating a new UserCollection
2021-03-05 17:01:39,130 users.py:27  INFO New UserCollection instance created
2021-03-05 17:01:39,130 main.py:36  INFO Createing a new UserStatusCollection
2021-03-05 17:01:39,231 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:01:39,231 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:01:39,231 main.py:67  ERROR accounts.csv does not contain data
2021-03-05 17:01:39,231 main.py:158 INFO Adding a new User
2021-03-05 17:01:39,574 users.py:45  INFO User dave03 sucessfully added
2021-03-05 17:01:39,575 main.py:158 INFO Adding a new User
2021-03-05 17:01:39,577 users.py:43  ERROR User dave03 breaks database rule: E11000 duplicate key error collection: social.UserTable index: user_id_1 dup key: { user_id: "dave03" }, full error: {'index': 0, 'code': 11000, 'keyPattern': {'user_id': 1}, 'keyValue': {'user_id': 'dave03'}, 'errmsg': 'E11000 duplicate key error collection: social.UserTable index: user_id_1 dup key: { user_id: "dave03" }'}
2021-03-05 17:01:39,577 main.py:170 INFO Updating an User
2021-03-05 17:01:39,637 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:01:39,640 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:01:39,641 user_status.py:45  INFO Status rando_001 added to database
2021-03-05 17:01:39,643 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:01:39,643 user_status.py:43  ERROR Status rando_001 already exists in database
2021-03-05 17:01:39,645 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:01:39,646 user_status.py:39  ERROR User rando does not exist
2021-03-05 17:01:39,648 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:01:39,648 user_status.py:62  INFO Status rando_001 sucessfully modified
2021-03-05 17:01:39,650 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:01:39,651 user_status.py:57  ERROR Status rando_001 not found database
2021-03-05 17:01:39,652 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:01:39,653 user_status.py:76  INFO Status rando_001 sucessfully deleted
2021-03-05 17:01:39,655 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:01:39,656 user_status.py:74  ERROR Status rando_001 not found database
2021-03-05 17:01:39,657 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:01:39,658 user_status.py:90  INFO Status rando_001 sucessfully found
2021-03-05 17:01:39,660 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:01:39,660 user_status.py:88  ERROR Status rando_001 not found database
2021-03-05 17:01:39,662 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:01:39,663 user_status.py:99  INFO Found statuses posted by rando
2021-03-05 17:01:39,664 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:01:39,665 user_status.py:107 INFO Found statuses matching best
2021-03-05 17:01:39,667 users.py:27  INFO New UserCollection instance created
2021-03-05 17:01:39,669 users.py:27  INFO New UserCollection instance created
2021-03-05 17:01:39,670 users.py:45  INFO User rando sucessfully added
2021-03-05 17:01:39,672 users.py:27  INFO New UserCollection instance created
2021-03-05 17:01:39,672 users.py:43  ERROR User rando breaks database rule: ERR_MSG
2021-03-05 17:01:39,674 users.py:27  INFO New UserCollection instance created
2021-03-05 17:01:39,674 users.py:65  INFO User rando sucessfully modified
2021-03-05 17:01:39,676 users.py:27  INFO New UserCollection instance created
2021-03-05 17:01:39,676 users.py:61  ERROR User rando not found in database
2021-03-05 17:01:39,679 users.py:27  INFO New UserCollection instance created
2021-03-05 17:01:39,680 users.py:83  INFO User rando sucessfully deleted
2021-03-05 17:01:39,681 users.py:27  INFO New UserCollection instance created
2021-03-05 17:01:39,682 users.py:78  ERROR User rando not found in database
2021-03-05 17:01:39,683 users.py:27  INFO New UserCollection instance created
2021-03-05 17:01:39,684 users.py:96  INFO User rando sucessfully found
2021-03-05 17:01:39,686 users.py:27  INFO New UserCollection instance created
2021-03-05 17:01:39,686 users.py:94  ERROR User rando not found in database
2021-03-05 17:01:39,696 users.py:27  INFO New UserCollection instance created
2021-03-05 17:01:39,696 users.py:104 ERROR User id ############################## is longer than 30
2021-03-05 17:01:39,696 users.py:107 ERROR User name ############################## is longer than 30
2021-03-05 17:01:39,696 users.py:112 ERROR User last name #################################################################################################### is longer than 100
2021-03-05 17:02:49,988 main.py:27  INFO Creating a new UserCollection
2021-03-05 17:02:49,993 users.py:27  INFO New UserCollection instance created
2021-03-05 17:02:49,993 main.py:36  INFO Createing a new UserStatusCollection
2021-03-05 17:02:49,996 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:02:58,274 main.py:27  INFO Creating a new UserCollection
2021-03-05 17:02:58,275 main.py:36  INFO Createing a new UserStatusCollection
2021-03-05 17:02:58,277 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:02:58,277 main.py:86  INFO Saving collection in database
2021-03-05 17:02:58,279 main.py:59  ERROR User files must be in csv format
2021-03-05 17:02:58,280 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:02:58,280 main.py:72  ERROR Users need four parameters
2021-03-05 17:02:58,280 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:02:58,280 main.py:72  ERROR Users need four parameters
2021-03-05 17:02:58,280 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:02:58,281 main.py:72  ERROR Users need four parameters
2021-03-05 17:02:58,281 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:02:58,281 main.py:72  ERROR Users need four parameters
2021-03-05 17:02:58,281 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:02:58,281 main.py:72  ERROR Users need four parameters
2021-03-05 17:02:58,282 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:02:58,282 main.py:77  ERROR Users can not have empty fields
2021-03-05 17:02:58,283 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:02:58,283 main.py:77  ERROR Users can not have empty fields
2021-03-05 17:02:58,283 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:02:58,283 main.py:77  ERROR Users can not have empty fields
2021-03-05 17:02:58,283 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:02:58,283 main.py:77  ERROR Users can not have empty fields
2021-03-05 17:02:58,285 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:02:58,285 main.py:67  ERROR accounts.csv does not contain data
2021-03-05 17:02:58,286 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:02:58,286 main.py:140 INFO Saving collection in database
2021-03-05 17:02:58,288 main.py:113 ERROR UserStatus files must be in csv format
2021-03-05 17:02:58,289 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:02:58,289 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 17:02:58,289 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:02:58,289 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 17:02:58,290 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:02:58,290 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 17:02:58,290 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:02:58,290 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 17:02:58,290 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:02:58,290 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 17:02:58,292 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:02:58,292 main.py:131 ERROR UserStatus can not have empty fields
2021-03-05 17:02:58,292 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:02:58,292 main.py:131 ERROR UserStatus can not have empty fields
2021-03-05 17:02:58,292 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:02:58,292 main.py:131 ERROR UserStatus can not have empty fields
2021-03-05 17:02:58,294 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:02:58,294 main.py:121 ERROR status_updates.csv does not contain data
2021-03-05 17:02:58,295 main.py:158 INFO Adding a new User
2021-03-05 17:02:58,296 main.py:170 INFO Updating an User
2021-03-05 17:02:58,297 main.py:183 INFO Deleting an User
2021-03-05 17:02:58,299 main.py:197 INFO Searching for an User
2021-03-05 17:02:58,300 main.py:197 INFO Searching for an User
2021-03-05 17:02:58,301 main.py:215 INFO Adding an UserStatus
2021-03-05 17:02:58,302 main.py:227 INFO Updating an UserStatus
2021-03-05 17:02:58,303 main.py:239 INFO Deleting an UserStatus
2021-03-05 17:02:58,305 main.py:252 INFO Searching for an UserStatus
2021-03-05 17:02:58,306 main.py:252 INFO Searching for an UserStatus
2021-03-05 17:02:58,307 main.py:263 INFO Searching for matching UserStatuses
2021-03-05 17:02:58,309 main.py:272 INFO Searching for UserStatuses matching string
2021-03-05 17:02:58,323 main.py:27  INFO Creating a new UserCollection
2021-03-05 17:02:58,405 users.py:27  INFO New UserCollection instance created
2021-03-05 17:02:58,405 main.py:36  INFO Createing a new UserStatusCollection
2021-03-05 17:02:58,504 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:02:58,505 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:02:58,505 main.py:67  ERROR accounts.csv does not contain data
2021-03-05 17:02:58,505 main.py:158 INFO Adding a new User
2021-03-05 17:02:58,830 users.py:45  INFO User dave03 sucessfully added
2021-03-05 17:02:58,830 main.py:158 INFO Adding a new User
2021-03-05 17:02:58,832 users.py:43  ERROR User dave03 breaks database rule: E11000 duplicate key error collection: social.UserTable index: user_id_1 dup key: { user_id: "dave03" }, full error: {'index': 0, 'code': 11000, 'keyPattern': {'user_id': 1}, 'keyValue': {'user_id': 'dave03'}, 'errmsg': 'E11000 duplicate key error collection: social.UserTable index: user_id_1 dup key: { user_id: "dave03" }'}
2021-03-05 17:02:58,832 main.py:170 INFO Updating an User
2021-03-05 17:02:58,888 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:02:58,890 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:02:58,892 user_status.py:45  INFO Status rando_001 added to database
2021-03-05 17:02:58,894 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:02:58,894 user_status.py:43  ERROR Status rando_001 already exists in database
2021-03-05 17:02:58,896 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:02:58,897 user_status.py:39  ERROR User rando does not exist
2021-03-05 17:02:58,899 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:02:58,899 user_status.py:62  INFO Status rando_001 sucessfully modified
2021-03-05 17:02:58,901 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:02:58,901 user_status.py:57  ERROR Status rando_001 not found database
2021-03-05 17:02:58,903 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:02:58,904 user_status.py:76  INFO Status rando_001 sucessfully deleted
2021-03-05 17:02:58,906 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:02:58,906 user_status.py:74  ERROR Status rando_001 not found database
2021-03-05 17:02:58,908 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:02:58,908 user_status.py:90  INFO Status rando_001 sucessfully found
2021-03-05 17:02:58,910 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:02:58,911 user_status.py:88  ERROR Status rando_001 not found database
2021-03-05 17:02:58,912 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:02:58,913 user_status.py:99  INFO Found statuses posted by rando
2021-03-05 17:02:58,915 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:02:58,915 user_status.py:107 INFO Found statuses matching best
2021-03-05 17:02:58,917 users.py:27  INFO New UserCollection instance created
2021-03-05 17:02:58,920 users.py:27  INFO New UserCollection instance created
2021-03-05 17:02:58,920 users.py:45  INFO User rando sucessfully added
2021-03-05 17:02:58,923 users.py:27  INFO New UserCollection instance created
2021-03-05 17:02:58,924 users.py:43  ERROR User rando breaks database rule: ERR_MSG
2021-03-05 17:02:58,925 users.py:27  INFO New UserCollection instance created
2021-03-05 17:02:58,926 users.py:65  INFO User rando sucessfully modified
2021-03-05 17:02:58,928 users.py:27  INFO New UserCollection instance created
2021-03-05 17:02:58,928 users.py:61  ERROR User rando not found in database
2021-03-05 17:02:58,930 users.py:27  INFO New UserCollection instance created
2021-03-05 17:02:58,931 users.py:83  INFO User rando sucessfully deleted
2021-03-05 17:02:58,933 users.py:27  INFO New UserCollection instance created
2021-03-05 17:02:58,933 users.py:78  ERROR User rando not found in database
2021-03-05 17:02:58,935 users.py:27  INFO New UserCollection instance created
2021-03-05 17:02:58,936 users.py:96  INFO User rando sucessfully found
2021-03-05 17:02:58,937 users.py:27  INFO New UserCollection instance created
2021-03-05 17:02:58,938 users.py:94  ERROR User rando not found in database
2021-03-05 17:02:58,940 users.py:27  INFO New UserCollection instance created
2021-03-05 17:02:58,940 users.py:104 ERROR User id ############################## is longer than 30
2021-03-05 17:02:58,940 users.py:107 ERROR User name ############################## is longer than 30
2021-03-05 17:02:58,940 users.py:112 ERROR User last name #################################################################################################### is longer than 100
2021-03-05 17:11:23,260 main.py:27  INFO Creating a new UserCollection
2021-03-05 17:11:23,261 main.py:36  INFO Createing a new UserStatusCollection
2021-03-05 17:11:23,262 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:11:23,262 main.py:86  INFO Saving collection in database
2021-03-05 17:11:23,264 main.py:59  ERROR User files must be in csv format
2021-03-05 17:11:23,265 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:11:23,265 main.py:72  ERROR Users need four parameters
2021-03-05 17:11:23,266 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:11:23,266 main.py:72  ERROR Users need four parameters
2021-03-05 17:11:23,266 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:11:23,266 main.py:72  ERROR Users need four parameters
2021-03-05 17:11:23,266 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:11:23,266 main.py:72  ERROR Users need four parameters
2021-03-05 17:11:23,266 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:11:23,267 main.py:72  ERROR Users need four parameters
2021-03-05 17:11:23,268 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:11:23,268 main.py:77  ERROR Users can not have empty fields
2021-03-05 17:11:23,268 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:11:23,268 main.py:77  ERROR Users can not have empty fields
2021-03-05 17:11:23,268 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:11:23,268 main.py:77  ERROR Users can not have empty fields
2021-03-05 17:11:23,269 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:11:23,269 main.py:77  ERROR Users can not have empty fields
2021-03-05 17:11:23,270 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:11:23,270 main.py:67  ERROR accounts.csv does not contain data
2021-03-05 17:11:23,271 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:11:23,272 main.py:140 INFO Saving collection in database
2021-03-05 17:11:23,273 main.py:113 ERROR UserStatus files must be in csv format
2021-03-05 17:11:23,274 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:11:23,275 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 17:11:23,275 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:11:23,275 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 17:11:23,275 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:11:23,275 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 17:11:23,275 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:11:23,275 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 17:11:23,275 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:11:23,276 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 17:11:23,277 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:11:23,277 main.py:131 ERROR UserStatus can not have empty fields
2021-03-05 17:11:23,277 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:11:23,277 main.py:131 ERROR UserStatus can not have empty fields
2021-03-05 17:11:23,278 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:11:23,278 main.py:131 ERROR UserStatus can not have empty fields
2021-03-05 17:11:23,279 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:11:23,279 main.py:121 ERROR status_updates.csv does not contain data
2021-03-05 17:11:23,280 main.py:158 INFO Adding a new User
2021-03-05 17:11:23,282 main.py:170 INFO Updating an User
2021-03-05 17:11:23,283 main.py:183 INFO Deleting an User
2021-03-05 17:11:23,284 main.py:197 INFO Searching for an User
2021-03-05 17:11:23,285 main.py:197 INFO Searching for an User
2021-03-05 17:11:23,286 main.py:215 INFO Adding an UserStatus
2021-03-05 17:11:23,287 main.py:227 INFO Updating an UserStatus
2021-03-05 17:11:23,289 main.py:239 INFO Deleting an UserStatus
2021-03-05 17:11:23,290 main.py:252 INFO Searching for an UserStatus
2021-03-05 17:11:23,292 main.py:252 INFO Searching for an UserStatus
2021-03-05 17:11:23,292 main.py:263 INFO Searching for matching UserStatuses
2021-03-05 17:11:23,295 main.py:272 INFO Searching for UserStatuses matching string
2021-03-05 17:11:23,399 main.py:27  INFO Creating a new UserCollection
2021-03-05 17:11:23,477 users.py:27  INFO New UserCollection instance created
2021-03-05 17:11:23,478 main.py:36  INFO Createing a new UserStatusCollection
2021-03-05 17:11:23,589 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:11:23,589 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:11:23,589 main.py:67  ERROR accounts.csv does not contain data
2021-03-05 17:11:23,590 main.py:158 INFO Adding a new User
2021-03-05 17:11:23,909 users.py:45  INFO User dave03 sucessfully added
2021-03-05 17:11:23,909 main.py:158 INFO Adding a new User
2021-03-05 17:11:23,911 users.py:43  ERROR User dave03 breaks database rule: E11000 duplicate key error collection: social.UserTable index: user_id_1 dup key: { user_id: "dave03" }, full error: {'index': 0, 'code': 11000, 'keyPattern': {'user_id': 1}, 'keyValue': {'user_id': 'dave03'}, 'errmsg': 'E11000 duplicate key error collection: social.UserTable index: user_id_1 dup key: { user_id: "dave03" }'}
2021-03-05 17:11:23,911 main.py:170 INFO Updating an User
2021-03-05 17:11:23,916 users.py:65  INFO User dave03 sucessfully modified
2021-03-05 17:11:23,917 main.py:170 INFO Updating an User
2021-03-05 17:11:23,918 users.py:61  ERROR User dave02 not found in database
2021-03-05 17:11:23,919 main.py:197 INFO Searching for an User
2021-03-05 17:11:23,920 users.py:96  INFO User dave03 sucessfully found
2021-03-05 17:11:23,920 main.py:197 INFO Searching for an User
2021-03-05 17:11:23,922 users.py:94  ERROR User dave02 not found in database
2021-03-05 17:11:23,922 main.py:183 INFO Deleting an User
2021-03-05 17:11:23,936 users.py:83  INFO User dave03 sucessfully deleted
2021-03-05 17:11:23,936 main.py:183 INFO Deleting an User
2021-03-05 17:11:23,938 users.py:78  ERROR User dave03 not found in database
2021-03-05 17:11:23,938 main.py:158 INFO Adding a new User
2021-03-05 17:11:23,940 users.py:45  INFO User dave03 sucessfully added
2021-03-05 17:11:23,940 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:11:23,940 main.py:121 ERROR status_updates.csv does not contain data
2021-03-05 17:11:23,940 main.py:215 INFO Adding an UserStatus
2021-03-05 17:11:24,014 user_status.py:45  INFO Status dave03_01 added to database
2021-03-05 17:11:24,014 main.py:215 INFO Adding an UserStatus
2021-03-05 17:11:24,018 user_status.py:43  ERROR Status dave03_01 already exists in database
2021-03-05 17:11:24,018 main.py:227 INFO Updating an UserStatus
2021-03-05 17:11:24,021 user_status.py:62  INFO Status dave03_01 sucessfully modified
2021-03-05 17:11:24,021 main.py:227 INFO Updating an UserStatus
2021-03-05 17:11:24,022 user_status.py:57  ERROR Status dave03_02 not found database
2021-03-05 17:11:24,022 main.py:252 INFO Searching for an UserStatus
2021-03-05 17:11:24,024 user_status.py:90  INFO Status dave03_01 sucessfully found
2021-03-05 17:11:24,025 main.py:252 INFO Searching for an UserStatus
2021-03-05 17:11:24,026 user_status.py:88  ERROR Status dave03 not found database
2021-03-05 17:11:24,026 main.py:239 INFO Deleting an UserStatus
2021-03-05 17:11:24,028 user_status.py:76  INFO Status dave03_01 sucessfully deleted
2021-03-05 17:11:24,028 main.py:239 INFO Deleting an UserStatus
2021-03-05 17:11:24,030 user_status.py:74  ERROR Status dave03_01 not found database
2021-03-05 17:11:24,030 main.py:215 INFO Adding an UserStatus
2021-03-05 17:11:24,034 user_status.py:45  INFO Status dave03_01 added to database
2021-03-05 17:11:24,034 main.py:183 INFO Deleting an User
2021-03-05 17:11:24,037 users.py:83  INFO User dave03 sucessfully deleted
2021-03-05 17:11:24,037 main.py:252 INFO Searching for an UserStatus
2021-03-05 17:11:24,039 user_status.py:88  ERROR Status dave03_01 not found database
2021-03-05 17:11:24,039 main.py:158 INFO Adding a new User
2021-03-05 17:11:24,039 users.py:104 ERROR User id ############################## is longer than 30
2021-03-05 17:11:24,039 main.py:158 INFO Adding a new User
2021-03-05 17:11:24,039 users.py:107 ERROR User name ############################## is longer than 30
2021-03-05 17:11:24,039 main.py:158 INFO Adding a new User
2021-03-05 17:11:24,039 users.py:112 ERROR User last name #################################################################################################### is longer than 100
2021-03-05 17:11:24,042 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:11:24,045 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:11:24,046 user_status.py:45  INFO Status rando_001 added to database
2021-03-05 17:11:24,048 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:11:24,049 user_status.py:43  ERROR Status rando_001 already exists in database
2021-03-05 17:11:24,050 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:11:24,051 user_status.py:39  ERROR User rando does not exist
2021-03-05 17:11:24,054 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:11:24,055 user_status.py:62  INFO Status rando_001 sucessfully modified
2021-03-05 17:11:24,098 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:11:24,098 user_status.py:57  ERROR Status rando_001 not found database
2021-03-05 17:11:24,100 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:11:24,101 user_status.py:76  INFO Status rando_001 sucessfully deleted
2021-03-05 17:11:24,102 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:11:24,103 user_status.py:74  ERROR Status rando_001 not found database
2021-03-05 17:11:24,104 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:11:24,105 user_status.py:90  INFO Status rando_001 sucessfully found
2021-03-05 17:11:24,107 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:11:24,107 user_status.py:88  ERROR Status rando_001 not found database
2021-03-05 17:11:24,109 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:11:24,110 user_status.py:99  INFO Found statuses posted by rando
2021-03-05 17:11:24,111 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:11:24,112 user_status.py:107 INFO Found statuses matching best
2021-03-05 17:11:24,114 users.py:27  INFO New UserCollection instance created
2021-03-05 17:11:24,116 users.py:27  INFO New UserCollection instance created
2021-03-05 17:11:24,117 users.py:45  INFO User rando sucessfully added
2021-03-05 17:11:24,118 users.py:27  INFO New UserCollection instance created
2021-03-05 17:11:24,119 users.py:43  ERROR User rando breaks database rule: ERR_MSG
2021-03-05 17:11:24,121 users.py:27  INFO New UserCollection instance created
2021-03-05 17:11:24,121 users.py:65  INFO User rando sucessfully modified
2021-03-05 17:11:24,147 users.py:27  INFO New UserCollection instance created
2021-03-05 17:11:24,147 users.py:61  ERROR User rando not found in database
2021-03-05 17:11:24,149 users.py:27  INFO New UserCollection instance created
2021-03-05 17:11:24,150 users.py:83  INFO User rando sucessfully deleted
2021-03-05 17:11:24,160 users.py:27  INFO New UserCollection instance created
2021-03-05 17:11:24,161 users.py:78  ERROR User rando not found in database
2021-03-05 17:11:24,163 users.py:27  INFO New UserCollection instance created
2021-03-05 17:11:24,163 users.py:96  INFO User rando sucessfully found
2021-03-05 17:11:24,165 users.py:27  INFO New UserCollection instance created
2021-03-05 17:11:24,165 users.py:94  ERROR User rando not found in database
2021-03-05 17:11:24,167 users.py:27  INFO New UserCollection instance created
2021-03-05 17:11:24,167 users.py:104 ERROR User id ############################## is longer than 30
2021-03-05 17:11:24,167 users.py:107 ERROR User name ############################## is longer than 30
2021-03-05 17:11:24,167 users.py:112 ERROR User last name #################################################################################################### is longer than 100
2021-03-05 17:14:46,359 main.py:27  INFO Creating a new UserCollection
2021-03-05 17:14:46,360 main.py:36  INFO Createing a new UserStatusCollection
2021-03-05 17:14:46,362 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:14:46,362 main.py:86  INFO Saving collection in database
2021-03-05 17:14:46,364 main.py:59  ERROR User files must be in csv format
2021-03-05 17:14:46,365 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:14:46,365 main.py:72  ERROR Users need four parameters
2021-03-05 17:14:46,365 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:14:46,366 main.py:72  ERROR Users need four parameters
2021-03-05 17:14:46,366 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:14:46,366 main.py:72  ERROR Users need four parameters
2021-03-05 17:14:46,366 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:14:46,366 main.py:72  ERROR Users need four parameters
2021-03-05 17:14:46,366 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:14:46,366 main.py:72  ERROR Users need four parameters
2021-03-05 17:14:46,368 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:14:46,368 main.py:77  ERROR Users can not have empty fields
2021-03-05 17:14:46,368 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:14:46,368 main.py:77  ERROR Users can not have empty fields
2021-03-05 17:14:46,368 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:14:46,368 main.py:77  ERROR Users can not have empty fields
2021-03-05 17:14:46,368 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:14:46,368 main.py:77  ERROR Users can not have empty fields
2021-03-05 17:14:46,370 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:14:46,370 main.py:67  ERROR accounts.csv does not contain data
2021-03-05 17:14:46,371 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:14:46,371 main.py:140 INFO Saving collection in database
2021-03-05 17:14:46,373 main.py:113 ERROR UserStatus files must be in csv format
2021-03-05 17:14:46,375 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:14:46,375 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 17:14:46,375 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:14:46,375 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 17:14:46,375 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:14:46,375 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 17:14:46,375 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:14:46,376 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 17:14:46,376 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:14:46,376 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 17:14:46,377 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:14:46,377 main.py:131 ERROR UserStatus can not have empty fields
2021-03-05 17:14:46,377 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:14:46,377 main.py:131 ERROR UserStatus can not have empty fields
2021-03-05 17:14:46,378 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:14:46,378 main.py:131 ERROR UserStatus can not have empty fields
2021-03-05 17:14:46,384 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:14:46,384 main.py:121 ERROR status_updates.csv does not contain data
2021-03-05 17:14:46,385 main.py:158 INFO Adding a new User
2021-03-05 17:14:46,386 main.py:170 INFO Updating an User
2021-03-05 17:14:46,387 main.py:183 INFO Deleting an User
2021-03-05 17:14:46,389 main.py:197 INFO Searching for an User
2021-03-05 17:14:46,390 main.py:197 INFO Searching for an User
2021-03-05 17:14:46,391 main.py:215 INFO Adding an UserStatus
2021-03-05 17:14:46,392 main.py:227 INFO Updating an UserStatus
2021-03-05 17:14:46,393 main.py:239 INFO Deleting an UserStatus
2021-03-05 17:14:46,395 main.py:252 INFO Searching for an UserStatus
2021-03-05 17:14:46,396 main.py:252 INFO Searching for an UserStatus
2021-03-05 17:14:46,397 main.py:263 INFO Searching for matching UserStatuses
2021-03-05 17:14:46,399 main.py:272 INFO Searching for UserStatuses matching string
2021-03-05 17:14:46,500 main.py:27  INFO Creating a new UserCollection
2021-03-05 17:14:46,579 users.py:27  INFO New UserCollection instance created
2021-03-05 17:14:46,579 main.py:36  INFO Createing a new UserStatusCollection
2021-03-05 17:14:46,700 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:14:46,701 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:14:46,701 main.py:67  ERROR accounts.csv does not contain data
2021-03-05 17:14:46,701 main.py:158 INFO Adding a new User
2021-03-05 17:14:47,019 users.py:45  INFO User dave03 sucessfully added
2021-03-05 17:14:47,020 main.py:158 INFO Adding a new User
2021-03-05 17:14:47,022 users.py:43  ERROR User dave03 breaks database rule: E11000 duplicate key error collection: social.UserTable index: user_id_1 dup key: { user_id: "dave03" }, full error: {'index': 0, 'code': 11000, 'keyPattern': {'user_id': 1}, 'keyValue': {'user_id': 'dave03'}, 'errmsg': 'E11000 duplicate key error collection: social.UserTable index: user_id_1 dup key: { user_id: "dave03" }'}
2021-03-05 17:14:47,022 main.py:170 INFO Updating an User
2021-03-05 17:14:47,025 users.py:65  INFO User dave03 sucessfully modified
2021-03-05 17:14:47,025 main.py:170 INFO Updating an User
2021-03-05 17:14:47,027 users.py:61  ERROR User dave02 not found in database
2021-03-05 17:14:47,027 main.py:197 INFO Searching for an User
2021-03-05 17:14:47,029 users.py:96  INFO User dave03 sucessfully found
2021-03-05 17:14:47,029 main.py:197 INFO Searching for an User
2021-03-05 17:14:47,031 users.py:94  ERROR User dave02 not found in database
2021-03-05 17:14:47,031 main.py:183 INFO Deleting an User
2021-03-05 17:14:47,051 users.py:83  INFO User dave03 sucessfully deleted
2021-03-05 17:14:47,051 main.py:183 INFO Deleting an User
2021-03-05 17:14:47,053 users.py:78  ERROR User dave03 not found in database
2021-03-05 17:14:47,053 main.py:158 INFO Adding a new User
2021-03-05 17:14:47,055 users.py:45  INFO User dave03 sucessfully added
2021-03-05 17:14:47,056 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:14:47,056 main.py:121 ERROR status_updates.csv does not contain data
2021-03-05 17:14:47,056 main.py:215 INFO Adding an UserStatus
2021-03-05 17:14:47,111 user_status.py:45  INFO Status dave03_01 added to database
2021-03-05 17:14:47,112 main.py:215 INFO Adding an UserStatus
2021-03-05 17:14:47,115 user_status.py:43  ERROR Status dave03_01 already exists in database
2021-03-05 17:14:47,116 main.py:227 INFO Updating an UserStatus
2021-03-05 17:14:47,118 user_status.py:62  INFO Status dave03_01 sucessfully modified
2021-03-05 17:14:47,118 main.py:227 INFO Updating an UserStatus
2021-03-05 17:14:47,120 user_status.py:57  ERROR Status dave03_02 not found database
2021-03-05 17:14:47,121 main.py:252 INFO Searching for an UserStatus
2021-03-05 17:14:47,123 user_status.py:90  INFO Status dave03_01 sucessfully found
2021-03-05 17:14:47,123 main.py:252 INFO Searching for an UserStatus
2021-03-05 17:14:47,125 user_status.py:88  ERROR Status dave03 not found database
2021-03-05 17:14:47,125 main.py:239 INFO Deleting an UserStatus
2021-03-05 17:14:47,127 user_status.py:76  INFO Status dave03_01 sucessfully deleted
2021-03-05 17:14:47,127 main.py:239 INFO Deleting an UserStatus
2021-03-05 17:14:47,129 user_status.py:74  ERROR Status dave03_01 not found database
2021-03-05 17:14:47,129 main.py:215 INFO Adding an UserStatus
2021-03-05 17:14:47,133 user_status.py:45  INFO Status dave03_01 added to database
2021-03-05 17:14:47,133 main.py:183 INFO Deleting an User
2021-03-05 17:14:47,137 users.py:83  INFO User dave03 sucessfully deleted
2021-03-05 17:14:47,137 main.py:252 INFO Searching for an UserStatus
2021-03-05 17:14:47,139 user_status.py:88  ERROR Status dave03_01 not found database
2021-03-05 17:14:47,139 main.py:158 INFO Adding a new User
2021-03-05 17:14:47,139 users.py:104 ERROR User id ############################## is longer than 30
2021-03-05 17:14:47,139 main.py:158 INFO Adding a new User
2021-03-05 17:14:47,139 users.py:107 ERROR User name ############################## is longer than 30
2021-03-05 17:14:47,140 main.py:158 INFO Adding a new User
2021-03-05 17:14:47,140 users.py:112 ERROR User last name #################################################################################################### is longer than 100
2021-03-05 17:14:47,142 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:14:47,145 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:14:47,146 user_status.py:45  INFO Status rando_001 added to database
2021-03-05 17:14:47,148 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:14:47,149 user_status.py:43  ERROR Status rando_001 already exists in database
2021-03-05 17:14:47,151 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:14:47,152 user_status.py:39  ERROR User rando does not exist
2021-03-05 17:14:47,153 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:14:47,154 user_status.py:62  INFO Status rando_001 sucessfully modified
2021-03-05 17:14:47,156 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:14:47,156 user_status.py:57  ERROR Status rando_001 not found database
2021-03-05 17:14:47,158 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:14:47,159 user_status.py:76  INFO Status rando_001 sucessfully deleted
2021-03-05 17:14:47,160 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:14:47,161 user_status.py:74  ERROR Status rando_001 not found database
2021-03-05 17:14:47,163 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:14:47,163 user_status.py:90  INFO Status rando_001 sucessfully found
2021-03-05 17:14:47,165 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:14:47,165 user_status.py:88  ERROR Status rando_001 not found database
2021-03-05 17:14:47,167 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:14:47,168 user_status.py:99  INFO Found statuses posted by rando
2021-03-05 17:14:47,169 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:14:47,170 user_status.py:107 INFO Found statuses matching best
2021-03-05 17:14:47,172 users.py:27  INFO New UserCollection instance created
2021-03-05 17:14:47,174 users.py:27  INFO New UserCollection instance created
2021-03-05 17:14:47,175 users.py:45  INFO User rando sucessfully added
2021-03-05 17:14:47,176 users.py:27  INFO New UserCollection instance created
2021-03-05 17:14:47,177 users.py:43  ERROR User rando breaks database rule: ERR_MSG
2021-03-05 17:14:47,178 users.py:27  INFO New UserCollection instance created
2021-03-05 17:14:47,179 users.py:65  INFO User rando sucessfully modified
2021-03-05 17:14:47,181 users.py:27  INFO New UserCollection instance created
2021-03-05 17:14:47,181 users.py:61  ERROR User rando not found in database
2021-03-05 17:14:47,183 users.py:27  INFO New UserCollection instance created
2021-03-05 17:14:47,184 users.py:83  INFO User rando sucessfully deleted
2021-03-05 17:14:47,186 users.py:27  INFO New UserCollection instance created
2021-03-05 17:14:47,186 users.py:78  ERROR User rando not found in database
2021-03-05 17:14:47,188 users.py:27  INFO New UserCollection instance created
2021-03-05 17:14:47,188 users.py:96  INFO User rando sucessfully found
2021-03-05 17:14:47,190 users.py:27  INFO New UserCollection instance created
2021-03-05 17:14:47,191 users.py:94  ERROR User rando not found in database
2021-03-05 17:14:47,192 users.py:27  INFO New UserCollection instance created
2021-03-05 17:14:47,192 users.py:104 ERROR User id ############################## is longer than 30
2021-03-05 17:14:47,193 users.py:107 ERROR User name ############################## is longer than 30
2021-03-05 17:14:47,193 users.py:112 ERROR User last name #################################################################################################### is longer than 100
2021-03-05 17:18:32,472 main.py:27  INFO Creating a new UserCollection
2021-03-05 17:18:32,474 main.py:36  INFO Createing a new UserStatusCollection
2021-03-05 17:18:32,475 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:18:32,476 main.py:86  INFO Saving collection in database
2021-03-05 17:18:32,477 main.py:59  ERROR User files must be in csv format
2021-03-05 17:18:32,479 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:18:32,479 main.py:72  ERROR Users need four parameters
2021-03-05 17:18:32,479 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:18:32,479 main.py:72  ERROR Users need four parameters
2021-03-05 17:18:32,479 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:18:32,479 main.py:72  ERROR Users need four parameters
2021-03-05 17:18:32,480 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:18:32,480 main.py:72  ERROR Users need four parameters
2021-03-05 17:18:32,480 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:18:32,480 main.py:72  ERROR Users need four parameters
2021-03-05 17:18:32,482 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:18:32,482 main.py:77  ERROR Users can not have empty fields
2021-03-05 17:18:32,482 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:18:32,482 main.py:77  ERROR Users can not have empty fields
2021-03-05 17:18:32,482 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:18:32,482 main.py:77  ERROR Users can not have empty fields
2021-03-05 17:18:32,482 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:18:32,483 main.py:77  ERROR Users can not have empty fields
2021-03-05 17:18:32,484 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:18:32,484 main.py:67  ERROR accounts.csv does not contain data
2021-03-05 17:18:32,485 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:18:32,485 main.py:140 INFO Saving collection in database
2021-03-05 17:18:32,487 main.py:113 ERROR UserStatus files must be in csv format
2021-03-05 17:18:32,488 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:18:32,488 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 17:18:32,489 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:18:32,489 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 17:18:32,489 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:18:32,489 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 17:18:32,489 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:18:32,489 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 17:18:32,489 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:18:32,489 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 17:18:32,491 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:18:32,491 main.py:131 ERROR UserStatus can not have empty fields
2021-03-05 17:18:32,491 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:18:32,491 main.py:131 ERROR UserStatus can not have empty fields
2021-03-05 17:18:32,491 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:18:32,491 main.py:131 ERROR UserStatus can not have empty fields
2021-03-05 17:18:32,493 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:18:32,493 main.py:121 ERROR status_updates.csv does not contain data
2021-03-05 17:18:32,494 main.py:158 INFO Adding a new User
2021-03-05 17:18:32,495 main.py:170 INFO Updating an User
2021-03-05 17:18:32,496 main.py:183 INFO Deleting an User
2021-03-05 17:18:32,498 main.py:197 INFO Searching for an User
2021-03-05 17:18:32,499 main.py:197 INFO Searching for an User
2021-03-05 17:18:32,500 main.py:215 INFO Adding an UserStatus
2021-03-05 17:18:32,502 main.py:227 INFO Updating an UserStatus
2021-03-05 17:18:32,508 main.py:239 INFO Deleting an UserStatus
2021-03-05 17:18:32,509 main.py:252 INFO Searching for an UserStatus
2021-03-05 17:18:32,510 main.py:252 INFO Searching for an UserStatus
2021-03-05 17:18:32,512 main.py:263 INFO Searching for matching UserStatuses
2021-03-05 17:18:32,513 main.py:272 INFO Searching for UserStatuses matching string
2021-03-05 17:18:32,602 main.py:27  INFO Creating a new UserCollection
2021-03-05 17:18:32,670 users.py:27  INFO New UserCollection instance created
2021-03-05 17:18:32,671 main.py:36  INFO Createing a new UserStatusCollection
2021-03-05 17:18:32,771 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:18:32,771 main.py:62  INFO Loading users from accounts.csv
2021-03-05 17:18:32,771 main.py:67  ERROR accounts.csv does not contain data
2021-03-05 17:18:32,772 main.py:158 INFO Adding a new User
2021-03-05 17:18:33,122 users.py:45  INFO User dave03 sucessfully added
2021-03-05 17:18:33,122 main.py:158 INFO Adding a new User
2021-03-05 17:18:33,124 users.py:43  ERROR User dave03 breaks database rule: E11000 duplicate key error collection: social.UserTable index: user_id_1 dup key: { user_id: "dave03" }, full error: {'index': 0, 'code': 11000, 'keyPattern': {'user_id': 1}, 'keyValue': {'user_id': 'dave03'}, 'errmsg': 'E11000 duplicate key error collection: social.UserTable index: user_id_1 dup key: { user_id: "dave03" }'}
2021-03-05 17:18:33,124 main.py:170 INFO Updating an User
2021-03-05 17:18:33,127 users.py:65  INFO User dave03 sucessfully modified
2021-03-05 17:18:33,127 main.py:170 INFO Updating an User
2021-03-05 17:18:33,129 users.py:61  ERROR User dave02 not found in database
2021-03-05 17:18:33,130 main.py:197 INFO Searching for an User
2021-03-05 17:18:33,132 users.py:96  INFO User dave03 sucessfully found
2021-03-05 17:18:33,132 main.py:197 INFO Searching for an User
2021-03-05 17:18:33,134 users.py:94  ERROR User dave02 not found in database
2021-03-05 17:18:33,134 main.py:183 INFO Deleting an User
2021-03-05 17:18:33,137 users.py:83  INFO User dave03 sucessfully deleted
2021-03-05 17:18:33,137 main.py:183 INFO Deleting an User
2021-03-05 17:18:33,139 users.py:78  ERROR User dave03 not found in database
2021-03-05 17:18:33,139 main.py:158 INFO Adding a new User
2021-03-05 17:18:33,141 users.py:45  INFO User dave03 sucessfully added
2021-03-05 17:18:33,141 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 17:18:33,141 main.py:121 ERROR status_updates.csv does not contain data
2021-03-05 17:18:33,141 main.py:215 INFO Adding an UserStatus
2021-03-05 17:18:33,184 user_status.py:45  INFO Status dave03_01 added to database
2021-03-05 17:18:33,184 main.py:215 INFO Adding an UserStatus
2021-03-05 17:18:33,189 user_status.py:43  ERROR Status dave03_01 already exists in database
2021-03-05 17:18:33,189 main.py:227 INFO Updating an UserStatus
2021-03-05 17:18:33,191 user_status.py:62  INFO Status dave03_01 sucessfully modified
2021-03-05 17:18:33,191 main.py:227 INFO Updating an UserStatus
2021-03-05 17:18:33,193 user_status.py:57  ERROR Status dave03_02 not found database
2021-03-05 17:18:33,193 main.py:252 INFO Searching for an UserStatus
2021-03-05 17:18:33,195 user_status.py:90  INFO Status dave03_01 sucessfully found
2021-03-05 17:18:33,195 main.py:252 INFO Searching for an UserStatus
2021-03-05 17:18:33,198 user_status.py:88  ERROR Status dave03 not found database
2021-03-05 17:18:33,198 main.py:239 INFO Deleting an UserStatus
2021-03-05 17:18:33,200 user_status.py:76  INFO Status dave03_01 sucessfully deleted
2021-03-05 17:18:33,200 main.py:239 INFO Deleting an UserStatus
2021-03-05 17:18:33,202 user_status.py:74  ERROR Status dave03_01 not found database
2021-03-05 17:18:33,202 main.py:215 INFO Adding an UserStatus
2021-03-05 17:18:33,206 user_status.py:45  INFO Status dave03_01 added to database
2021-03-05 17:18:33,206 main.py:183 INFO Deleting an User
2021-03-05 17:18:33,209 users.py:83  INFO User dave03 sucessfully deleted
2021-03-05 17:18:33,209 main.py:252 INFO Searching for an UserStatus
2021-03-05 17:18:33,211 user_status.py:88  ERROR Status dave03_01 not found database
2021-03-05 17:18:33,211 main.py:158 INFO Adding a new User
2021-03-05 17:18:33,211 users.py:105 ERROR User id ############################## is longer than 30
2021-03-05 17:18:33,211 main.py:158 INFO Adding a new User
2021-03-05 17:18:33,211 users.py:109 ERROR User name ############################## is longer than 30
2021-03-05 17:18:33,211 main.py:158 INFO Adding a new User
2021-03-05 17:18:33,211 users.py:113 ERROR User last name #################################################################################################### is longer than 100
2021-03-05 17:18:33,214 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:18:33,216 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:18:33,217 user_status.py:45  INFO Status rando_001 added to database
2021-03-05 17:18:33,220 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:18:33,221 user_status.py:43  ERROR Status rando_001 already exists in database
2021-03-05 17:18:33,223 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:18:33,224 user_status.py:39  ERROR User rando does not exist
2021-03-05 17:18:33,225 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:18:33,226 user_status.py:62  INFO Status rando_001 sucessfully modified
2021-03-05 17:18:33,228 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:18:33,228 user_status.py:57  ERROR Status rando_001 not found database
2021-03-05 17:18:33,230 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:18:33,231 user_status.py:76  INFO Status rando_001 sucessfully deleted
2021-03-05 17:18:33,232 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:18:33,233 user_status.py:74  ERROR Status rando_001 not found database
2021-03-05 17:18:33,235 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:18:33,236 user_status.py:90  INFO Status rando_001 sucessfully found
2021-03-05 17:18:33,237 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:18:33,238 user_status.py:88  ERROR Status rando_001 not found database
2021-03-05 17:18:33,239 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:18:33,240 user_status.py:99  INFO Found statuses posted by rando
2021-03-05 17:18:33,242 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 17:18:33,242 user_status.py:107 INFO Found statuses matching best
2021-03-05 17:18:33,245 users.py:27  INFO New UserCollection instance created
2021-03-05 17:18:33,247 users.py:27  INFO New UserCollection instance created
2021-03-05 17:18:33,247 users.py:45  INFO User rando sucessfully added
2021-03-05 17:18:33,249 users.py:27  INFO New UserCollection instance created
2021-03-05 17:18:33,249 users.py:43  ERROR User rando breaks database rule: ERR_MSG
2021-03-05 17:18:33,251 users.py:27  INFO New UserCollection instance created
2021-03-05 17:18:33,252 users.py:65  INFO User rando sucessfully modified
2021-03-05 17:18:33,254 users.py:27  INFO New UserCollection instance created
2021-03-05 17:18:33,254 users.py:61  ERROR User rando not found in database
2021-03-05 17:18:33,256 users.py:27  INFO New UserCollection instance created
2021-03-05 17:18:33,257 users.py:83  INFO User rando sucessfully deleted
2021-03-05 17:18:33,259 users.py:27  INFO New UserCollection instance created
2021-03-05 17:18:33,259 users.py:78  ERROR User rando not found in database
2021-03-05 17:18:33,261 users.py:27  INFO New UserCollection instance created
2021-03-05 17:18:33,262 users.py:96  INFO User rando sucessfully found
2021-03-05 17:18:33,263 users.py:27  INFO New UserCollection instance created
2021-03-05 17:18:33,264 users.py:94  ERROR User rando not found in database
2021-03-05 17:18:33,265 users.py:27  INFO New UserCollection instance created
2021-03-05 17:18:33,266 users.py:105 ERROR User id ############################## is longer than 30
2021-03-05 17:18:33,266 users.py:109 ERROR User name ############################## is longer than 30
2021-03-05 17:18:33,266 users.py:113 ERROR User last name #################################################################################################### is longer than 100
2021-03-05 19:28:12,380 main.py:27  INFO Creating a new UserCollection
2021-03-05 19:28:12,382 main.py:36  INFO Createing a new UserStatusCollection
2021-03-05 19:28:12,384 main.py:62  INFO Loading users from accounts.csv
2021-03-05 19:28:12,384 main.py:86  INFO Saving collection in database
2021-03-05 19:28:12,387 main.py:59  ERROR User files must be in csv format
2021-03-05 19:28:12,389 main.py:62  INFO Loading users from accounts.csv
2021-03-05 19:28:12,389 main.py:72  ERROR Users need four parameters
2021-03-05 19:28:12,389 main.py:62  INFO Loading users from accounts.csv
2021-03-05 19:28:12,389 main.py:72  ERROR Users need four parameters
2021-03-05 19:28:12,389 main.py:62  INFO Loading users from accounts.csv
2021-03-05 19:28:12,389 main.py:72  ERROR Users need four parameters
2021-03-05 19:28:12,390 main.py:62  INFO Loading users from accounts.csv
2021-03-05 19:28:12,390 main.py:72  ERROR Users need four parameters
2021-03-05 19:28:12,390 main.py:62  INFO Loading users from accounts.csv
2021-03-05 19:28:12,390 main.py:72  ERROR Users need four parameters
2021-03-05 19:28:12,392 main.py:62  INFO Loading users from accounts.csv
2021-03-05 19:28:12,392 main.py:77  ERROR Users can not have empty fields
2021-03-05 19:28:12,392 main.py:62  INFO Loading users from accounts.csv
2021-03-05 19:28:12,392 main.py:77  ERROR Users can not have empty fields
2021-03-05 19:28:12,392 main.py:62  INFO Loading users from accounts.csv
2021-03-05 19:28:12,393 main.py:77  ERROR Users can not have empty fields
2021-03-05 19:28:12,393 main.py:62  INFO Loading users from accounts.csv
2021-03-05 19:28:12,393 main.py:77  ERROR Users can not have empty fields
2021-03-05 19:28:12,395 main.py:62  INFO Loading users from accounts.csv
2021-03-05 19:28:12,395 main.py:67  ERROR accounts.csv does not contain data
2021-03-05 19:28:12,397 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 19:28:12,397 main.py:140 INFO Saving collection in database
2021-03-05 19:28:12,399 main.py:113 ERROR UserStatus files must be in csv format
2021-03-05 19:28:12,401 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 19:28:12,401 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 19:28:12,401 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 19:28:12,401 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 19:28:12,402 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 19:28:12,402 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 19:28:12,402 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 19:28:12,402 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 19:28:12,402 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 19:28:12,402 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 19:28:12,404 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 19:28:12,404 main.py:131 ERROR UserStatus can not have empty fields
2021-03-05 19:28:12,405 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 19:28:12,405 main.py:131 ERROR UserStatus can not have empty fields
2021-03-05 19:28:12,405 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 19:28:12,405 main.py:131 ERROR UserStatus can not have empty fields
2021-03-05 19:28:12,407 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 19:28:12,407 main.py:121 ERROR status_updates.csv does not contain data
2021-03-05 19:28:12,409 main.py:158 INFO Adding a new User
2021-03-05 19:28:12,411 main.py:170 INFO Updating an User
2021-03-05 19:28:12,419 main.py:183 INFO Deleting an User
2021-03-05 19:28:12,421 main.py:197 INFO Searching for an User
2021-03-05 19:28:12,422 main.py:197 INFO Searching for an User
2021-03-05 19:28:12,424 main.py:215 INFO Adding an UserStatus
2021-03-05 19:28:12,426 main.py:227 INFO Updating an UserStatus
2021-03-05 19:28:12,427 main.py:239 INFO Deleting an UserStatus
2021-03-05 19:28:12,429 main.py:252 INFO Searching for an UserStatus
2021-03-05 19:28:12,431 main.py:252 INFO Searching for an UserStatus
2021-03-05 19:28:12,433 main.py:263 INFO Searching for matching UserStatuses
2021-03-05 19:28:12,435 main.py:272 INFO Searching for UserStatuses matching string
2021-03-05 19:28:32,985 main.py:27  INFO Creating a new UserCollection
2021-03-05 19:28:33,063 users.py:27  INFO New UserCollection instance created
2021-03-05 19:28:33,063 main.py:36  INFO Createing a new UserStatusCollection
2021-03-05 19:28:33,152 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 19:28:33,153 main.py:62  INFO Loading users from accounts.csv
2021-03-05 19:28:33,157 main.py:86  INFO Saving collection in database
2021-03-05 19:28:33,512 main.py:158 INFO Adding a new User
2021-03-05 19:28:33,515 users.py:45  INFO User dave03 sucessfully added
2021-03-05 19:28:33,515 main.py:158 INFO Adding a new User
2021-03-05 19:28:33,518 users.py:43  ERROR User dave03 breaks database rule: E11000 duplicate key error collection: social.UserTable index: user_id_1 dup key: { user_id: "dave03" }, full error: {'index': 0, 'code': 11000, 'keyPattern': {'user_id': 1}, 'keyValue': {'user_id': 'dave03'}, 'errmsg': 'E11000 duplicate key error collection: social.UserTable index: user_id_1 dup key: { user_id: "dave03" }'}
2021-03-05 19:28:33,518 main.py:170 INFO Updating an User
2021-03-05 19:28:33,521 users.py:65  INFO User dave03 sucessfully modified
2021-03-05 19:28:33,521 main.py:170 INFO Updating an User
2021-03-05 19:28:33,523 users.py:61  ERROR User dave02 not found in database
2021-03-05 19:28:33,524 main.py:197 INFO Searching for an User
2021-03-05 19:28:33,526 users.py:96  INFO User dave03 sucessfully found
2021-03-05 19:28:33,527 main.py:197 INFO Searching for an User
2021-03-05 19:28:33,529 users.py:94  ERROR User dave02 not found in database
2021-03-05 19:28:33,529 main.py:183 INFO Deleting an User
2021-03-05 19:28:33,534 users.py:83  INFO User dave03 sucessfully deleted
2021-03-05 19:28:33,534 main.py:183 INFO Deleting an User
2021-03-05 19:28:33,536 users.py:78  ERROR User dave03 not found in database
2021-03-05 19:28:33,536 main.py:158 INFO Adding a new User
2021-03-05 19:28:33,539 users.py:45  INFO User dave03 sucessfully added
2021-03-05 19:28:33,539 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 19:28:33,928 main.py:140 INFO Saving collection in database
2021-03-05 19:28:37,702 main.py:215 INFO Adding an UserStatus
2021-03-05 19:28:37,708 user_status.py:45  INFO Status dave03_01 added to database
2021-03-05 19:28:37,708 main.py:215 INFO Adding an UserStatus
2021-03-05 19:28:37,712 user_status.py:43  ERROR Status dave03_01 already exists in database
2021-03-05 19:28:37,713 main.py:227 INFO Updating an UserStatus
2021-03-05 19:28:37,716 user_status.py:62  INFO Status dave03_01 sucessfully modified
2021-03-05 19:28:37,716 main.py:227 INFO Updating an UserStatus
2021-03-05 19:28:37,718 user_status.py:57  ERROR Status dave03_02 not found database
2021-03-05 19:28:37,719 main.py:252 INFO Searching for an UserStatus
2021-03-05 19:28:37,721 user_status.py:90  INFO Status dave03_01 sucessfully found
2021-03-05 19:28:37,722 main.py:252 INFO Searching for an UserStatus
2021-03-05 19:28:37,724 user_status.py:88  ERROR Status dave03 not found database
2021-03-05 19:28:37,724 main.py:239 INFO Deleting an UserStatus
2021-03-05 19:28:37,727 user_status.py:76  INFO Status dave03_01 sucessfully deleted
2021-03-05 19:28:37,727 main.py:239 INFO Deleting an UserStatus
2021-03-05 19:28:37,730 user_status.py:74  ERROR Status dave03_01 not found database
2021-03-05 19:28:37,730 main.py:215 INFO Adding an UserStatus
2021-03-05 19:28:37,735 user_status.py:45  INFO Status dave03_01 added to database
2021-03-05 19:28:37,735 main.py:183 INFO Deleting an User
2021-03-05 19:28:37,784 users.py:83  INFO User dave03 sucessfully deleted
2021-03-05 19:28:37,784 main.py:252 INFO Searching for an UserStatus
2021-03-05 19:28:37,786 user_status.py:88  ERROR Status dave03_01 not found database
2021-03-05 19:28:37,787 main.py:158 INFO Adding a new User
2021-03-05 19:28:37,787 users.py:105 ERROR User id ############################## is longer than 30
2021-03-05 19:28:37,787 main.py:158 INFO Adding a new User
2021-03-05 19:28:37,787 users.py:109 ERROR User name ############################## is longer than 30
2021-03-05 19:28:37,787 main.py:158 INFO Adding a new User
2021-03-05 19:28:37,787 users.py:113 ERROR User last name #################################################################################################### is longer than 100
2021-03-05 19:30:16,337 main.py:27  INFO Creating a new UserCollection
2021-03-05 19:30:16,339 main.py:36  INFO Createing a new UserStatusCollection
2021-03-05 19:30:16,340 main.py:62  INFO Loading users from accounts.csv
2021-03-05 19:30:16,341 main.py:86  INFO Saving collection in database
2021-03-05 19:30:16,343 main.py:59  ERROR User files must be in csv format
2021-03-05 19:30:16,345 main.py:62  INFO Loading users from accounts.csv
2021-03-05 19:30:16,345 main.py:72  ERROR Users need four parameters
2021-03-05 19:30:16,345 main.py:62  INFO Loading users from accounts.csv
2021-03-05 19:30:16,345 main.py:72  ERROR Users need four parameters
2021-03-05 19:30:16,345 main.py:62  INFO Loading users from accounts.csv
2021-03-05 19:30:16,345 main.py:72  ERROR Users need four parameters
2021-03-05 19:30:16,346 main.py:62  INFO Loading users from accounts.csv
2021-03-05 19:30:16,346 main.py:72  ERROR Users need four parameters
2021-03-05 19:30:16,346 main.py:62  INFO Loading users from accounts.csv
2021-03-05 19:30:16,346 main.py:72  ERROR Users need four parameters
2021-03-05 19:30:16,348 main.py:62  INFO Loading users from accounts.csv
2021-03-05 19:30:16,348 main.py:77  ERROR Users can not have empty fields
2021-03-05 19:30:16,348 main.py:62  INFO Loading users from accounts.csv
2021-03-05 19:30:16,348 main.py:77  ERROR Users can not have empty fields
2021-03-05 19:30:16,349 main.py:62  INFO Loading users from accounts.csv
2021-03-05 19:30:16,349 main.py:77  ERROR Users can not have empty fields
2021-03-05 19:30:16,349 main.py:62  INFO Loading users from accounts.csv
2021-03-05 19:30:16,349 main.py:77  ERROR Users can not have empty fields
2021-03-05 19:30:16,351 main.py:62  INFO Loading users from accounts.csv
2021-03-05 19:30:16,351 main.py:67  ERROR accounts.csv does not contain data
2021-03-05 19:30:16,353 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 19:30:16,353 main.py:140 INFO Saving collection in database
2021-03-05 19:30:16,355 main.py:113 ERROR UserStatus files must be in csv format
2021-03-05 19:30:16,357 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 19:30:16,357 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 19:30:16,357 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 19:30:16,357 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 19:30:16,357 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 19:30:16,358 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 19:30:16,358 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 19:30:16,358 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 19:30:16,358 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 19:30:16,358 main.py:126 ERROR UserStatus needs three parameters
2021-03-05 19:30:16,360 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 19:30:16,360 main.py:131 ERROR UserStatus can not have empty fields
2021-03-05 19:30:16,361 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 19:30:16,361 main.py:131 ERROR UserStatus can not have empty fields
2021-03-05 19:30:16,361 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 19:30:16,361 main.py:131 ERROR UserStatus can not have empty fields
2021-03-05 19:30:16,368 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 19:30:16,368 main.py:121 ERROR status_updates.csv does not contain data
2021-03-05 19:30:16,370 main.py:158 INFO Adding a new User
2021-03-05 19:30:16,372 main.py:170 INFO Updating an User
2021-03-05 19:30:16,374 main.py:183 INFO Deleting an User
2021-03-05 19:30:16,375 main.py:197 INFO Searching for an User
2021-03-05 19:30:16,377 main.py:197 INFO Searching for an User
2021-03-05 19:30:16,378 main.py:215 INFO Adding an UserStatus
2021-03-05 19:30:16,380 main.py:227 INFO Updating an UserStatus
2021-03-05 19:30:16,382 main.py:239 INFO Deleting an UserStatus
2021-03-05 19:30:16,383 main.py:252 INFO Searching for an UserStatus
2021-03-05 19:30:16,386 main.py:252 INFO Searching for an UserStatus
2021-03-05 19:30:16,387 main.py:263 INFO Searching for matching UserStatuses
2021-03-05 19:30:16,389 main.py:272 INFO Searching for UserStatuses matching string
2021-03-05 19:30:16,484 main.py:27  INFO Creating a new UserCollection
2021-03-05 19:30:16,562 users.py:27  INFO New UserCollection instance created
2021-03-05 19:30:16,563 main.py:36  INFO Createing a new UserStatusCollection
2021-03-05 19:30:16,651 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 19:30:16,652 main.py:62  INFO Loading users from accounts.csv
2021-03-05 19:30:16,652 main.py:67  ERROR accounts.csv does not contain data
2021-03-05 19:30:16,652 main.py:158 INFO Adding a new User
2021-03-05 19:30:17,005 users.py:45  INFO User dave03 sucessfully added
2021-03-05 19:30:17,005 main.py:158 INFO Adding a new User
2021-03-05 19:30:17,008 users.py:43  ERROR User dave03 breaks database rule: E11000 duplicate key error collection: social.UserTable index: user_id_1 dup key: { user_id: "dave03" }, full error: {'index': 0, 'code': 11000, 'keyPattern': {'user_id': 1}, 'keyValue': {'user_id': 'dave03'}, 'errmsg': 'E11000 duplicate key error collection: social.UserTable index: user_id_1 dup key: { user_id: "dave03" }'}
2021-03-05 19:30:17,008 main.py:170 INFO Updating an User
2021-03-05 19:30:17,011 users.py:65  INFO User dave03 sucessfully modified
2021-03-05 19:30:17,011 main.py:170 INFO Updating an User
2021-03-05 19:30:17,014 users.py:61  ERROR User dave02 not found in database
2021-03-05 19:30:17,014 main.py:197 INFO Searching for an User
2021-03-05 19:30:17,016 users.py:96  INFO User dave03 sucessfully found
2021-03-05 19:30:17,017 main.py:197 INFO Searching for an User
2021-03-05 19:30:17,019 users.py:94  ERROR User dave02 not found in database
2021-03-05 19:30:17,019 main.py:183 INFO Deleting an User
2021-03-05 19:30:17,024 users.py:83  INFO User dave03 sucessfully deleted
2021-03-05 19:30:17,024 main.py:183 INFO Deleting an User
2021-03-05 19:30:17,026 users.py:78  ERROR User dave03 not found in database
2021-03-05 19:30:17,026 main.py:158 INFO Adding a new User
2021-03-05 19:30:17,029 users.py:45  INFO User dave03 sucessfully added
2021-03-05 19:30:17,029 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 19:30:17,029 main.py:121 ERROR status_updates.csv does not contain data
2021-03-05 19:30:17,029 main.py:215 INFO Adding an UserStatus
2021-03-05 19:30:17,082 user_status.py:45  INFO Status dave03_01 added to database
2021-03-05 19:30:17,082 main.py:215 INFO Adding an UserStatus
2021-03-05 19:30:17,086 user_status.py:43  ERROR Status dave03_01 already exists in database
2021-03-05 19:30:17,087 main.py:227 INFO Updating an UserStatus
2021-03-05 19:30:17,090 user_status.py:62  INFO Status dave03_01 sucessfully modified
2021-03-05 19:30:17,090 main.py:227 INFO Updating an UserStatus
2021-03-05 19:30:17,093 user_status.py:57  ERROR Status dave03_02 not found database
2021-03-05 19:30:17,093 main.py:252 INFO Searching for an UserStatus
2021-03-05 19:30:17,095 user_status.py:90  INFO Status dave03_01 sucessfully found
2021-03-05 19:30:17,096 main.py:252 INFO Searching for an UserStatus
2021-03-05 19:30:17,098 user_status.py:88  ERROR Status dave03 not found database
2021-03-05 19:30:17,098 main.py:239 INFO Deleting an UserStatus
2021-03-05 19:30:17,101 user_status.py:76  INFO Status dave03_01 sucessfully deleted
2021-03-05 19:30:17,101 main.py:239 INFO Deleting an UserStatus
2021-03-05 19:30:17,103 user_status.py:74  ERROR Status dave03_01 not found database
2021-03-05 19:30:17,103 main.py:215 INFO Adding an UserStatus
2021-03-05 19:30:17,108 user_status.py:45  INFO Status dave03_01 added to database
2021-03-05 19:30:17,108 main.py:183 INFO Deleting an User
2021-03-05 19:30:17,112 users.py:83  INFO User dave03 sucessfully deleted
2021-03-05 19:30:17,113 main.py:252 INFO Searching for an UserStatus
2021-03-05 19:30:17,115 user_status.py:88  ERROR Status dave03_01 not found database
2021-03-05 19:30:17,115 main.py:158 INFO Adding a new User
2021-03-05 19:30:17,116 users.py:105 ERROR User id ############################## is longer than 30
2021-03-05 19:30:17,116 main.py:158 INFO Adding a new User
2021-03-05 19:30:17,116 users.py:109 ERROR User name ############################## is longer than 30
2021-03-05 19:30:17,116 main.py:158 INFO Adding a new User
2021-03-05 19:30:17,116 users.py:113 ERROR User last name #################################################################################################### is longer than 100
2021-03-05 19:30:17,119 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 19:30:17,124 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 19:30:17,126 user_status.py:45  INFO Status rando_001 added to database
2021-03-05 19:30:17,128 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 19:30:17,129 user_status.py:43  ERROR Status rando_001 already exists in database
2021-03-05 19:30:17,132 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 19:30:17,133 user_status.py:39  ERROR User rando does not exist
2021-03-05 19:30:17,135 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 19:30:17,136 user_status.py:62  INFO Status rando_001 sucessfully modified
2021-03-05 19:30:17,139 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 19:30:17,139 user_status.py:57  ERROR Status rando_001 not found database
2021-03-05 19:30:17,142 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 19:30:17,143 user_status.py:76  INFO Status rando_001 sucessfully deleted
2021-03-05 19:30:17,145 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 19:30:17,146 user_status.py:74  ERROR Status rando_001 not found database
2021-03-05 19:30:17,148 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 19:30:17,149 user_status.py:90  INFO Status rando_001 sucessfully found
2021-03-05 19:30:17,151 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 19:30:17,152 user_status.py:88  ERROR Status rando_001 not found database
2021-03-05 19:30:17,155 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 19:30:17,156 user_status.py:99  INFO Found statuses posted by rando
2021-03-05 19:30:17,158 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 19:30:17,159 user_status.py:107 INFO Found statuses matching best
2021-03-05 19:30:17,162 users.py:27  INFO New UserCollection instance created
2021-03-05 19:30:17,165 users.py:27  INFO New UserCollection instance created
2021-03-05 19:30:17,166 users.py:45  INFO User rando sucessfully added
2021-03-05 19:30:17,168 users.py:27  INFO New UserCollection instance created
2021-03-05 19:30:17,169 users.py:43  ERROR User rando breaks database rule: ERR_MSG
2021-03-05 19:30:17,171 users.py:27  INFO New UserCollection instance created
2021-03-05 19:30:17,172 users.py:105 ERROR User id ############################## is longer than 30
2021-03-05 19:30:17,174 users.py:27  INFO New UserCollection instance created
2021-03-05 19:30:17,175 users.py:65  INFO User rando sucessfully modified
2021-03-05 19:30:17,178 users.py:27  INFO New UserCollection instance created
2021-03-05 19:30:17,178 users.py:61  ERROR User rando not found in database
2021-03-05 19:30:17,181 users.py:27  INFO New UserCollection instance created
2021-03-05 19:30:17,181 users.py:105 ERROR User id ############################## is longer than 30
2021-03-05 19:30:17,183 users.py:27  INFO New UserCollection instance created
2021-03-05 19:30:17,185 users.py:83  INFO User rando sucessfully deleted
2021-03-05 19:30:17,188 users.py:27  INFO New UserCollection instance created
2021-03-05 19:30:17,188 users.py:78  ERROR User rando not found in database
2021-03-05 19:30:17,192 users.py:27  INFO New UserCollection instance created
2021-03-05 19:30:17,192 users.py:96  INFO User rando sucessfully found
2021-03-05 19:30:17,195 users.py:27  INFO New UserCollection instance created
2021-03-05 19:30:17,195 users.py:94  ERROR User rando not found in database
2021-03-05 19:30:17,198 users.py:27  INFO New UserCollection instance created
2021-03-05 19:30:17,198 users.py:105 ERROR User id ############################## is longer than 30
2021-03-05 19:30:17,198 users.py:109 ERROR User name ############################## is longer than 30
2021-03-05 19:30:17,198 users.py:113 ERROR User last name #################################################################################################### is longer than 100
2021-03-05 19:36:29,315 main.py:27  INFO Creating a new UserCollection
2021-03-05 19:36:29,320 users.py:27  INFO New UserCollection instance created
2021-03-05 19:36:29,320 main.py:36  INFO Createing a new UserStatusCollection
2021-03-05 19:36:29,323 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 19:36:43,693 main.py:59  ERROR User files must be in csv format
2021-03-05 19:37:12,682 main.py:116 INFO Saving statuses to status_updates.csv
2021-03-05 19:37:12,895 main.py:140 INFO Saving collection in database
2021-03-05 19:37:25,953 main.py:62  INFO Loading users from accounts.csv
2021-03-05 19:37:25,956 main.py:86  INFO Saving collection in database
2021-03-05 19:37:33,957 main.py:62  INFO Loading users from accounts.csv
2021-03-05 19:37:33,960 main.py:86  INFO Saving collection in database
2021-03-05 19:46:38,046 main.py:27  INFO Creating a new UserCollection
2021-03-05 19:46:38,051 users.py:27  INFO New UserCollection instance created
2021-03-05 19:46:38,051 main.py:36  INFO Createing a new UserStatusCollection
2021-03-05 19:46:38,055 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 19:46:48,679 main.py:62  INFO Loading users from accounts.csv
2021-03-05 19:46:48,682 main.py:86  INFO Saving collection in database
2021-03-05 19:47:54,580 main.py:28  INFO Creating a new UserCollection
2021-03-05 19:47:54,586 users.py:27  INFO New UserCollection instance created
2021-03-05 19:47:54,586 main.py:37  INFO Createing a new UserStatusCollection
2021-03-05 19:47:54,589 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 19:47:58,513 main.py:63  INFO Loading users from accounts.csv
2021-03-05 19:47:58,516 main.py:87  INFO Saving collection in database
2021-03-05 19:47:58,530 main.py:93  INFO Bulk Write attempted to load duplicate users, ignored
2021-03-05 19:48:42,687 main.py:28  INFO Creating a new UserCollection
2021-03-05 19:48:42,692 users.py:27  INFO New UserCollection instance created
2021-03-05 19:48:42,692 main.py:37  INFO Createing a new UserStatusCollection
2021-03-05 19:48:42,695 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 19:49:00,979 main.py:120 INFO Saving statuses to status_updates.csv
2021-03-05 19:49:01,192 main.py:144 INFO Saving collection in database
2021-03-05 19:49:01,840 main.py:150 INFO Bulk Write attempted to load duplicate statuses, ignored
2021-03-05 19:49:09,679 main.py:120 INFO Saving statuses to status_updates.csv
2021-03-05 19:49:09,891 main.py:144 INFO Saving collection in database
2021-03-05 19:49:10,532 main.py:150 INFO Bulk Write attempted to load duplicate statuses, ignored
2021-03-05 19:49:18,737 main.py:279 INFO Searching for UserStatuses matching string
2021-03-05 19:49:18,737 user_status.py:107 INFO Found statuses matching have
2021-03-05 19:49:24,168 main.py:279 INFO Searching for UserStatuses matching string
2021-03-05 19:49:24,168 user_status.py:107 INFO Found statuses matching wine
2021-03-05 19:49:43,680 main.py:270 INFO Searching for matching UserStatuses
2021-03-05 19:49:43,725 user_status.py:99  INFO Found statuses posted by Turdi.Karlise38
2021-03-05 19:50:18,711 main.py:270 INFO Searching for matching UserStatuses
2021-03-05 19:50:18,757 user_status.py:99  INFO Found statuses posted by Trudi.Karlise38
2021-03-05 19:50:51,646 main.py:279 INFO Searching for UserStatuses matching string
2021-03-05 19:50:51,646 user_status.py:107 INFO Found statuses matching owner
2021-03-05 19:50:57,619 main.py:246 INFO Deleting an UserStatus
2021-03-05 19:50:57,622 user_status.py:76  INFO Status Estele.Strauss99_308 sucessfully deleted
2021-03-05 19:51:05,899 main.py:246 INFO Deleting an UserStatus
2021-03-05 19:51:05,902 user_status.py:76  INFO Status Lou.Gervais53_183 sucessfully deleted
2021-03-05 19:51:07,271 main.py:246 INFO Deleting an UserStatus
2021-03-05 19:51:07,274 user_status.py:76  INFO Status Jewel.Wachtel81_124 sucessfully deleted
2021-03-05 19:51:08,342 main.py:246 INFO Deleting an UserStatus
2021-03-05 19:51:08,344 user_status.py:76  INFO Status Loria.Morell64_931 sucessfully deleted
2021-03-05 19:51:09,264 main.py:246 INFO Deleting an UserStatus
2021-03-05 19:51:09,266 user_status.py:76  INFO Status Eva.Grissom3_572 sucessfully deleted
2021-03-05 19:51:09,902 main.py:246 INFO Deleting an UserStatus
2021-03-05 19:51:09,904 user_status.py:76  INFO Status Dasya.Kulsrud5_730 sucessfully deleted
2021-03-05 19:51:10,524 main.py:246 INFO Deleting an UserStatus
2021-03-05 19:51:10,526 user_status.py:76  INFO Status Kalindi.Hosbein70_568 sucessfully deleted
2021-03-05 19:51:11,084 main.py:246 INFO Deleting an UserStatus
2021-03-05 19:51:11,086 user_status.py:76  INFO Status Micaela.Le73_445 sucessfully deleted
2021-03-05 19:51:11,661 main.py:246 INFO Deleting an UserStatus
2021-03-05 19:51:11,664 user_status.py:76  INFO Status Rozina.Elsie80_205 sucessfully deleted
2021-03-05 19:51:12,203 main.py:246 INFO Deleting an UserStatus
2021-03-05 19:51:12,205 user_status.py:76  INFO Status Minerva.Wilbert87_208 sucessfully deleted
2021-03-05 19:51:12,793 main.py:246 INFO Deleting an UserStatus
2021-03-05 19:51:12,795 user_status.py:76  INFO Status Maxy.Mozelle87_393 sucessfully deleted
2021-03-05 19:51:13,337 main.py:246 INFO Deleting an UserStatus
2021-03-05 19:51:13,340 user_status.py:76  INFO Status Lynnett.Vernen34_233 sucessfully deleted
2021-03-05 19:51:13,916 main.py:246 INFO Deleting an UserStatus
2021-03-05 19:51:13,918 user_status.py:76  INFO Status Ronny.Burgener36_779 sucessfully deleted
2021-03-05 19:51:14,537 main.py:246 INFO Deleting an UserStatus
2021-03-05 19:51:14,539 user_status.py:76  INFO Status Rhoda.Terti59_533 sucessfully deleted
2021-03-05 19:51:15,113 main.py:246 INFO Deleting an UserStatus
2021-03-05 19:51:15,115 user_status.py:76  INFO Status Anna-Maria.Drusus57_411 sucessfully deleted
2021-03-05 19:51:15,670 main.py:246 INFO Deleting an UserStatus
2021-03-05 19:51:15,672 user_status.py:76  INFO Status Brooke.Fante62_717 sucessfully deleted
2021-03-05 19:51:16,264 main.py:246 INFO Deleting an UserStatus
2021-03-05 19:51:16,266 user_status.py:76  INFO Status Arluene.Cutlor89_337 sucessfully deleted
2021-03-05 19:51:16,840 main.py:246 INFO Deleting an UserStatus
2021-03-05 19:51:16,843 user_status.py:76  INFO Status Darci.Pitzer65_151 sucessfully deleted
2021-03-05 19:51:17,430 main.py:246 INFO Deleting an UserStatus
2021-03-05 19:51:17,432 user_status.py:76  INFO Status Lyn.Ehudd89_380 sucessfully deleted
2021-03-05 19:51:18,005 main.py:246 INFO Deleting an UserStatus
2021-03-05 19:51:18,007 user_status.py:76  INFO Status Donny.Atkins39_107 sucessfully deleted
2021-03-05 19:51:18,579 main.py:246 INFO Deleting an UserStatus
2021-03-05 19:51:18,581 user_status.py:76  INFO Status Mame.Kalle16_507 sucessfully deleted
2021-03-05 19:51:19,167 main.py:246 INFO Deleting an UserStatus
2021-03-05 19:51:19,169 user_status.py:76  INFO Status Estele.Savill70_347 sucessfully deleted
2021-03-05 19:51:19,746 main.py:246 INFO Deleting an UserStatus
2021-03-05 19:51:19,749 user_status.py:76  INFO Status Noella.Cacilie74_615 sucessfully deleted
2021-03-05 19:51:20,334 main.py:246 INFO Deleting an UserStatus
2021-03-05 19:51:20,337 user_status.py:76  INFO Status Hatti.Zeeba49_81 sucessfully deleted
2021-03-05 19:51:20,926 main.py:246 INFO Deleting an UserStatus
2021-03-05 19:51:20,929 user_status.py:76  INFO Status Zorina.Jeannie66_901 sucessfully deleted
2021-03-05 19:51:21,502 main.py:246 INFO Deleting an UserStatus
2021-03-05 19:51:21,504 user_status.py:76  INFO Status Carlina.Chae37_672 sucessfully deleted
2021-03-05 19:51:22,092 main.py:246 INFO Deleting an UserStatus
2021-03-05 19:51:22,094 user_status.py:76  INFO Status Paulina.Adalie26_820 sucessfully deleted
2021-03-05 19:51:24,037 main.py:246 INFO Deleting an UserStatus
2021-03-05 19:51:24,039 user_status.py:76  INFO Status Carol-Jean.Skiest89_843 sucessfully deleted
2021-03-05 19:51:45,149 main.py:28  INFO Creating a new UserCollection
2021-03-05 19:51:45,150 main.py:37  INFO Createing a new UserStatusCollection
2021-03-05 19:51:45,152 main.py:63  INFO Loading users from accounts.csv
2021-03-05 19:51:45,152 main.py:87  INFO Saving collection in database
2021-03-05 19:51:45,154 main.py:60  ERROR User files must be in csv format
2021-03-05 19:51:45,155 main.py:63  INFO Loading users from accounts.csv
2021-03-05 19:51:45,155 main.py:73  ERROR Users need four parameters
2021-03-05 19:51:45,155 main.py:63  INFO Loading users from accounts.csv
2021-03-05 19:51:45,155 main.py:73  ERROR Users need four parameters
2021-03-05 19:51:45,155 main.py:63  INFO Loading users from accounts.csv
2021-03-05 19:51:45,155 main.py:73  ERROR Users need four parameters
2021-03-05 19:51:45,156 main.py:63  INFO Loading users from accounts.csv
2021-03-05 19:51:45,156 main.py:73  ERROR Users need four parameters
2021-03-05 19:51:45,156 main.py:63  INFO Loading users from accounts.csv
2021-03-05 19:51:45,156 main.py:73  ERROR Users need four parameters
2021-03-05 19:51:45,157 main.py:63  INFO Loading users from accounts.csv
2021-03-05 19:51:45,157 main.py:78  ERROR Users can not have empty fields
2021-03-05 19:51:45,158 main.py:63  INFO Loading users from accounts.csv
2021-03-05 19:51:45,158 main.py:78  ERROR Users can not have empty fields
2021-03-05 19:51:45,158 main.py:63  INFO Loading users from accounts.csv
2021-03-05 19:51:45,158 main.py:78  ERROR Users can not have empty fields
2021-03-05 19:51:45,158 main.py:63  INFO Loading users from accounts.csv
2021-03-05 19:51:45,158 main.py:78  ERROR Users can not have empty fields
2021-03-05 19:51:45,159 main.py:63  INFO Loading users from accounts.csv
2021-03-05 19:51:45,160 main.py:68  ERROR accounts.csv does not contain data
2021-03-05 19:51:45,161 main.py:120 INFO Saving statuses to status_updates.csv
2021-03-05 19:51:45,161 main.py:144 INFO Saving collection in database
2021-03-05 19:51:45,163 main.py:117 ERROR UserStatus files must be in csv format
2021-03-05 19:51:45,164 main.py:120 INFO Saving statuses to status_updates.csv
2021-03-05 19:51:45,164 main.py:130 ERROR UserStatus needs three parameters
2021-03-05 19:51:45,164 main.py:120 INFO Saving statuses to status_updates.csv
2021-03-05 19:51:45,164 main.py:130 ERROR UserStatus needs three parameters
2021-03-05 19:51:45,164 main.py:120 INFO Saving statuses to status_updates.csv
2021-03-05 19:51:45,164 main.py:130 ERROR UserStatus needs three parameters
2021-03-05 19:51:45,165 main.py:120 INFO Saving statuses to status_updates.csv
2021-03-05 19:51:45,165 main.py:130 ERROR UserStatus needs three parameters
2021-03-05 19:51:45,165 main.py:120 INFO Saving statuses to status_updates.csv
2021-03-05 19:51:45,165 main.py:130 ERROR UserStatus needs three parameters
2021-03-05 19:51:45,167 main.py:120 INFO Saving statuses to status_updates.csv
2021-03-05 19:51:45,167 main.py:135 ERROR UserStatus can not have empty fields
2021-03-05 19:51:45,167 main.py:120 INFO Saving statuses to status_updates.csv
2021-03-05 19:51:45,167 main.py:135 ERROR UserStatus can not have empty fields
2021-03-05 19:51:45,167 main.py:120 INFO Saving statuses to status_updates.csv
2021-03-05 19:51:45,167 main.py:135 ERROR UserStatus can not have empty fields
2021-03-05 19:51:45,168 main.py:120 INFO Saving statuses to status_updates.csv
2021-03-05 19:51:45,169 main.py:125 ERROR status_updates.csv does not contain data
2021-03-05 19:51:45,170 main.py:165 INFO Adding a new User
2021-03-05 19:51:45,171 main.py:177 INFO Updating an User
2021-03-05 19:51:45,172 main.py:190 INFO Deleting an User
2021-03-05 19:51:45,173 main.py:204 INFO Searching for an User
2021-03-05 19:51:45,175 main.py:204 INFO Searching for an User
2021-03-05 19:51:45,176 main.py:222 INFO Adding an UserStatus
2021-03-05 19:51:45,177 main.py:234 INFO Updating an UserStatus
2021-03-05 19:51:45,178 main.py:246 INFO Deleting an UserStatus
2021-03-05 19:51:45,179 main.py:259 INFO Searching for an UserStatus
2021-03-05 19:51:45,181 main.py:259 INFO Searching for an UserStatus
2021-03-05 19:51:45,182 main.py:270 INFO Searching for matching UserStatuses
2021-03-05 19:51:45,183 main.py:279 INFO Searching for UserStatuses matching string
2021-03-05 19:51:45,197 main.py:28  INFO Creating a new UserCollection
2021-03-05 19:51:45,288 users.py:27  INFO New UserCollection instance created
2021-03-05 19:51:45,288 main.py:37  INFO Createing a new UserStatusCollection
2021-03-05 19:51:45,377 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 19:51:45,378 main.py:63  INFO Loading users from accounts.csv
2021-03-05 19:51:45,378 main.py:68  ERROR accounts.csv does not contain data
2021-03-05 19:51:45,378 main.py:165 INFO Adding a new User
2021-03-05 19:51:45,715 users.py:45  INFO User dave03 sucessfully added
2021-03-05 19:51:45,716 main.py:165 INFO Adding a new User
2021-03-05 19:51:45,718 users.py:43  ERROR User dave03 breaks database rule: E11000 duplicate key error collection: social.UserTable index: user_id_1 dup key: { user_id: "dave03" }, full error: {'index': 0, 'code': 11000, 'keyPattern': {'user_id': 1}, 'keyValue': {'user_id': 'dave03'}, 'errmsg': 'E11000 duplicate key error collection: social.UserTable index: user_id_1 dup key: { user_id: "dave03" }'}
2021-03-05 19:51:45,718 main.py:177 INFO Updating an User
2021-03-05 19:51:45,721 users.py:65  INFO User dave03 sucessfully modified
2021-03-05 19:51:45,722 main.py:177 INFO Updating an User
2021-03-05 19:51:45,724 users.py:61  ERROR User dave02 not found in database
2021-03-05 19:51:45,724 main.py:204 INFO Searching for an User
2021-03-05 19:51:45,726 users.py:96  INFO User dave03 sucessfully found
2021-03-05 19:51:45,726 main.py:204 INFO Searching for an User
2021-03-05 19:51:45,729 users.py:94  ERROR User dave02 not found in database
2021-03-05 19:51:45,729 main.py:190 INFO Deleting an User
2021-03-05 19:51:45,733 users.py:83  INFO User dave03 sucessfully deleted
2021-03-05 19:51:45,733 main.py:190 INFO Deleting an User
2021-03-05 19:51:45,735 users.py:78  ERROR User dave03 not found in database
2021-03-05 19:51:45,735 main.py:165 INFO Adding a new User
2021-03-05 19:51:45,737 users.py:45  INFO User dave03 sucessfully added
2021-03-05 19:51:45,738 main.py:120 INFO Saving statuses to status_updates.csv
2021-03-05 19:51:45,738 main.py:125 ERROR status_updates.csv does not contain data
2021-03-05 19:51:45,738 main.py:222 INFO Adding an UserStatus
2021-03-05 19:51:45,808 user_status.py:45  INFO Status dave03_01 added to database
2021-03-05 19:51:45,808 main.py:222 INFO Adding an UserStatus
2021-03-05 19:51:45,813 user_status.py:43  ERROR Status dave03_01 already exists in database
2021-03-05 19:51:45,813 main.py:234 INFO Updating an UserStatus
2021-03-05 19:51:45,815 user_status.py:62  INFO Status dave03_01 sucessfully modified
2021-03-05 19:51:45,816 main.py:234 INFO Updating an UserStatus
2021-03-05 19:51:45,818 user_status.py:57  ERROR Status dave03_02 not found database
2021-03-05 19:51:45,818 main.py:259 INFO Searching for an UserStatus
2021-03-05 19:51:45,820 user_status.py:90  INFO Status dave03_01 sucessfully found
2021-03-05 19:51:45,820 main.py:259 INFO Searching for an UserStatus
2021-03-05 19:51:45,822 user_status.py:88  ERROR Status dave03 not found database
2021-03-05 19:51:45,822 main.py:246 INFO Deleting an UserStatus
2021-03-05 19:51:45,825 user_status.py:76  INFO Status dave03_01 sucessfully deleted
2021-03-05 19:51:45,825 main.py:246 INFO Deleting an UserStatus
2021-03-05 19:51:45,826 user_status.py:74  ERROR Status dave03_01 not found database
2021-03-05 19:51:45,827 main.py:222 INFO Adding an UserStatus
2021-03-05 19:51:45,830 user_status.py:45  INFO Status dave03_01 added to database
2021-03-05 19:51:45,830 main.py:190 INFO Deleting an User
2021-03-05 19:51:45,834 users.py:83  INFO User dave03 sucessfully deleted
2021-03-05 19:51:45,834 main.py:259 INFO Searching for an UserStatus
2021-03-05 19:51:45,836 user_status.py:88  ERROR Status dave03_01 not found database
2021-03-05 19:51:45,836 main.py:165 INFO Adding a new User
2021-03-05 19:51:45,836 users.py:105 ERROR User id ############################## is longer than 30
2021-03-05 19:51:45,837 main.py:165 INFO Adding a new User
2021-03-05 19:51:45,837 users.py:109 ERROR User name ############################## is longer than 30
2021-03-05 19:51:45,837 main.py:165 INFO Adding a new User
2021-03-05 19:51:45,837 users.py:113 ERROR User last name #################################################################################################### is longer than 100
2021-03-05 19:51:45,839 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 19:51:45,843 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 19:51:45,844 user_status.py:45  INFO Status rando_001 added to database
2021-03-05 19:51:45,846 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 19:51:45,847 user_status.py:43  ERROR Status rando_001 already exists in database
2021-03-05 19:51:45,848 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 19:51:45,849 user_status.py:39  ERROR User rando does not exist
2021-03-05 19:51:45,851 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 19:51:45,852 user_status.py:62  INFO Status rando_001 sucessfully modified
2021-03-05 19:51:45,853 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 19:51:45,854 user_status.py:57  ERROR Status rando_001 not found database
2021-03-05 19:51:45,855 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 19:51:45,856 user_status.py:76  INFO Status rando_001 sucessfully deleted
2021-03-05 19:51:45,858 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 19:51:45,859 user_status.py:74  ERROR Status rando_001 not found database
2021-03-05 19:51:45,860 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 19:51:45,861 user_status.py:90  INFO Status rando_001 sucessfully found
2021-03-05 19:51:45,862 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 19:51:45,863 user_status.py:88  ERROR Status rando_001 not found database
2021-03-05 19:51:45,865 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 19:51:45,865 user_status.py:99  INFO Found statuses posted by rando
2021-03-05 19:51:45,867 user_status.py:23  INFO New UserStatusCollection instance created
2021-03-05 19:51:45,868 user_status.py:107 INFO Found statuses matching best
2021-03-05 19:51:45,869 users.py:27  INFO New UserCollection instance created
2021-03-05 19:51:45,872 users.py:27  INFO New UserCollection instance created
2021-03-05 19:51:45,873 users.py:45  INFO User rando sucessfully added
2021-03-05 19:51:45,874 users.py:27  INFO New UserCollection instance created
2021-03-05 19:51:45,875 users.py:43  ERROR User rando breaks database rule: ERR_MSG
2021-03-05 19:51:45,876 users.py:27  INFO New UserCollection instance created
2021-03-05 19:51:45,876 users.py:105 ERROR User id ############################## is longer than 30
2021-03-05 19:51:45,878 users.py:27  INFO New UserCollection instance created
2021-03-05 19:51:45,879 users.py:65  INFO User rando sucessfully modified
2021-03-05 19:51:45,881 users.py:27  INFO New UserCollection instance created
2021-03-05 19:51:45,881 users.py:61  ERROR User rando not found in database
2021-03-05 19:51:45,883 users.py:27  INFO New UserCollection instance created
2021-03-05 19:51:45,883 users.py:105 ERROR User id ############################## is longer than 30
2021-03-05 19:51:45,885 users.py:27  INFO New UserCollection instance created
2021-03-05 19:51:45,886 users.py:83  INFO User rando sucessfully deleted
2021-03-05 19:51:45,888 users.py:27  INFO New UserCollection instance created
2021-03-05 19:51:45,889 users.py:78  ERROR User rando not found in database
2021-03-05 19:51:45,890 users.py:27  INFO New UserCollection instance created
2021-03-05 19:51:45,891 users.py:96  INFO User rando sucessfully found
2021-03-05 19:51:45,892 users.py:27  INFO New UserCollection instance created
2021-03-05 19:51:45,893 users.py:94  ERROR User rando not found in database
2021-03-05 19:51:45,894 users.py:27  INFO New UserCollection instance created
2021-03-05 19:51:45,895 users.py:105 ERROR User id ############################## is longer than 30
2021-03-05 19:51:45,895 users.py:109 ERROR User name ############################## is longer than 30
2021-03-05 19:51:45,895 users.py:113 ERROR User last name #################################################################################################### is longer than 100
